In [1]:
from jqdatasdk import *
import configparser
 
config = configparser.ConfigParser()
file = 'config.ini'
config.read(file)
user = config.get('login','user')
password = config.get('login','password')
auth(user,password)

auth success （JQData现有流量增加活动，详情请咨询JQData管理员，微信号：JQData02）


# 行业概念相关API

提供行业板块信息，概念板块信息，包含行业代码、名称等。

- 获取行业类成份股请使用`get_industry_stocks(industry_code, date=None)`, 详情[见这里](https://www.joinquant.com/api#get_industry_stocks).
- 获取概念类成分股请使用`get_concept_stocks(concept_code, date=None)`, 详情[见这里](https://www.joinquant.com/api#get_concept_stocks).
- get_industries [获取行业列表](https://www.joinquant.com/help/api/help?name=api#get_industry_stocks)

- get_concepts [获取概念列表](https://www.joinquant.com/help/api/help?name=api#get_industry_stocks)

- get_industry [查询股票所属行业](https://www.joinquant.com/help/api/help?name=api#get_industry_stocks)

- get_concept [获取股票所属概念板块](https://www.joinquant.com/help/api/help?name=api#get_industry_stocks)


- 目前申万行业提供行情数据，频率为天，交易日17:40更新；
- [获取中证指数/行业的行情,pe,pb,股息率](https://www.joinquant.com/view/community/detail/16656)
- 概念板块目前没有行情数据；

(温馨提示：鉴于内容太多, 可使用`Ctrl + F`进行搜索)

## get_industry_stocks  获取行业成份股
`get_concept_stocks(concept_code, date=None)`  
获取在给定日期一个概念板块的所有股票，概念板块分类列表见数据页面-行业概念数据。

**参数**

- concept_code: 概念板块编码
- date: 查询日期, 一个字符串(格式类似'2015-10-15')或者[datetime.date]/[datetime.datetime]对象, 可以是None, 使用默认日期. 这个默认日期在回测和研究模块上有点差别:
 - 回测模块: 默认值会随着回测日期变化而变化, 等于context.current_dt
 - 研究模块: 默认是今天  
 
**返回** 返回股票代码的list

**示例**

In [4]:
# 获取计算机/互联网行业的成分股
stocks = get_industry_stocks('I64')
print(stocks)

['000503.XSHE', '000606.XSHE', '000676.XSHE', '000835.XSHE', '002072.XSHE', '002095.XSHE', '002113.XSHE', '002131.XSHE', '002137.XSHE', '002174.XSHE', '002175.XSHE', '002235.XSHE', '002247.XSHE', '002315.XSHE', '002354.XSHE', '002425.XSHE', '002439.XSHE', '002447.XSHE', '002464.XSHE', '002467.XSHE', '002517.XSHE', '002530.XSHE', '002555.XSHE', '002558.XSHE', '002619.XSHE', '002624.XSHE', '300031.XSHE', '300038.XSHE', '300043.XSHE', '300052.XSHE', '300059.XSHE', '300104.XSHE', '300113.XSHE', '300226.XSHE', '300242.XSHE', '300295.XSHE', '300315.XSHE', '300343.XSHE', '300392.XSHE', '300418.XSHE', '300431.XSHE', '300467.XSHE', '300494.XSHE', '300571.XSHE', '300766.XSHE', '300773.XSHE', '300785.XSHE', '600242.XSHG', '600358.XSHG', '600633.XSHG', '600634.XSHG', '600640.XSHG', '600652.XSHG', '600804.XSHG', '600899.XSHG', '600986.XSHG', '601360.XSHG', '603000.XSHG', '603258.XSHG', '603444.XSHG', '603533.XSHG', '603825.XSHG', '603881.XSHG', '603888.XSHG']


## get_concept_stocks  获取概念成份股
`get_concept_stocks(concept_code, date=None)`  
获取在给定日期一个概念板块的所有股票，概念板块分类列表见数据页面-行业概念数据。

**参数**

- concept_code: 概念板块编码
- date: 查询日期, 一个字符串(格式类似'2015-10-15')或者[datetime.date]/[datetime.datetime]对象, 可以是None, 使用默认日期. 这个默认日期在回测和研究模块上有点差别:
  - 回测模块: 默认值会随着回测日期变化而变化, 等于context.current_dt
  - 研究模块: 默认是今天
**返回** 返回股票代码的list

**示例**

In [2]:
# 获取风力发电概念板块的成分股
stocks = get_concept_stocks('GN036', date='2019-04-16')
print(stocks)

['000791.XSHE', '000836.XSHE', '000862.XSHE', '000958.XSHE', '002080.XSHE', '002202.XSHE', '002204.XSHE', '002531.XSHE', '300040.XSHE', '300129.XSHE', '300169.XSHE', '300185.XSHE', '300443.XSHE', '300569.XSHE', '300690.XSHE', '600163.XSHG', '600192.XSHG', '600290.XSHG', '600416.XSHG', '600458.XSHG', '600483.XSHG', '600875.XSHG', '601016.XSHG', '601218.XSHG', '601558.XSHG', '601619.XSHG', '603063.XSHG', '603693.XSHG']


## get_industries  获取行业列表
`get_industries(name)`  
按照行业分类获取行业列表。

**参数**

- name: 行业代码， 取值如下：
  - "sw_l1": 申万一级行业
  - "sw_l2": 申万二级行业
  - "sw_l3": 申万三级行业
  - "jq_l1": 聚宽一级行业
  - "jq_l2": 聚宽二级行业
  - "zjw": 证监会行业

**返回值**

- pandas.DataFrame， 各 column 的含义如下:
  - index: 行业代码
  - name: 行业名称
  - start_date: 开始日期

In [13]:
# 获取申万一级行业列表
get_industries(name='sw_l1').head()

,name,start_date
801740,国防军工I,2014-02-21
801020,采掘I,2004-02-10
801110,家用电器I,2004-02-10
801160,公用事业I,2004-02-10
801060,建筑建材I,2004-02-10


## get_concepts  获取概念列表
`get_concepts()`  
获取所有的概念板块列表，概念列表见数据页面-[行业概念数据](https://www.joinquant.com/data/dict/plateData)。

**返回值**

- pandas.DataFrame， 各 column 的含义如下:
  - index: 概念代码
  - name: 概念名称
  - start_date: 开始日期

In [8]:
# 获取概念列表
get_concepts().head()

,name,start_date
GN001,参股金融,2013-12-31
GN028,智能电网,2013-12-31
GN030,物联网,2013-12-31
GN031,重组,2013-12-31
GN032,迪士尼,2013-12-31


## get_industry 查询股票所属行业
`get_industry(security, date=None)`  
**参数**

- security：标的代码。类型为字符串，形式如"000001.XSHE"；或为包含标的代码字符串的列表，形如["000001.XSHE", "000002.XSHE"]
- date：查询的日期。类型为字符串，形如"2018-06-01"或"2018-06-01 09:00:00"；或为datetime.datetime对象和datetime.date。*注意传入对象的时分秒将被忽略*。

**返回结果**

一个dict， key是标的代码。

**示例**

In [11]:
get_industry(security=['000001.XSHE','000002.XSHE'], date="2018-06-01")

{'000001.XSHE': {'sw_l1': {'industry_code': '801780', 'industry_name': '银行I'},
  'sw_l2': {'industry_code': '801192', 'industry_name': '银行II'},
  'sw_l3': {'industry_code': '851911', 'industry_name': '银行III'},
  'zjw': {'industry_code': 'J66', 'industry_name': '货币金融服务'},
  'jq_l2': {'industry_code': 'HY493', 'industry_name': '多元化银行指数'},
  'jq_l1': {'industry_code': 'HY007', 'industry_name': '金融指数'}},
 '000002.XSHE': {'sw_l1': {'industry_code': '801180', 'industry_name': '房地产I'},
  'sw_l2': {'industry_code': '801181', 'industry_name': '房地产开发II'},
  'sw_l3': {'industry_code': '851811', 'industry_name': '房地产开发III'},
  'zjw': {'industry_code': 'K70', 'industry_name': '房地产业'},
  'jq_l2': {'industry_code': 'HY509', 'industry_name': '房地产开发指数'},
  'jq_l1': {'industry_code': 'HY011', 'industry_name': '房地产指数'}}}

In [15]:
# 将获取到的嵌套dict转为panel
import pandas as pd
panel = pd.Panel(get_industry(security=['000001.XSHE','000002.XSHE'], date="2018-06-01"))
panel

H:\anaconda\lib\site-packages\IPython\core\interactiveshell.py:2963: FutureWarning: 
Panel is deprecated and will be removed in a future version.
The recommended way to represent these types of 3-dimensional data are with a MultiIndex on a DataFrame, via the Panel.to_frame() method
Alternatively, you can use the xarray package http://xarray.pydata.org/en/stable/.
Pandas provides a `.to_xarray()` method to help automate this conversion.

  exec(code_obj, self.user_global_ns, self.user_ns)


<class 'pandas.core.panel.Panel'>
Dimensions: 2 (items) x 2 (major_axis) x 6 (minor_axis)
Items axis: 000001.XSHE to 000002.XSHE
Major_axis axis: industry_code to industry_name
Minor_axis axis: sw_l1 to jq_l1

In [21]:
# 提取panel中的数据
panel[:,:,'sw_l1']

,000001.XSHE,000002.XSHE
industry_code,801780,801180
industry_name,银行I,房地产I


## get_concept 获取股票所属概念板块(jqdatasdk即将上线)
`get_concept(security, date=None)`  
获取股票所属概念板块。返回一个dict，key为标的代码，value详见示例。

- security: 标的代码或标的列表

- date: 要查询的日期，日期字符串/date对象/datetime对象，注意传入datetime对象时忽略日内时间   
code是传入的股票代码；多个code返回嵌套的多个字典。 ‘jq_concept’代表聚宽概念； XX是code所在的概念代码； YY是code所在的概念名称。   

**示例**   

In [22]:
dict1 = get_concept('000001.XSHE', date='2019-07-15')
print(dict1)

NameError: name 'get_concept' is not defined

# **行业类**

## 证监会行业

| 行业代码 | 行业名称                                 | 起始日期   |
| -------- | ---------------------------------------- | ---------- |
| A01      | 农业                                     | 1993-09-17 |
| A02      | 林业                                     | 1996-12-06 |
| A03      | 畜牧业                                   | 1997-06-11 |
| A04      | 渔业                                     | 1993-05-07 |
| A05      | 农、林、牧、渔服务业                     | 1997-05-30 |
| B06      | 煤炭开采和洗选业                         | 1994-01-06 |
| B07      | 石油和天然气开采业                       | 1996-06-28 |
| B08      | 黑色金属矿采选业                         | 1997-07-08 |
| B09      | 有色金属矿采选业                         | 1996-03-20 |
| B10      | 非金属矿采选业                           | 2017-05-03 |
| B11      | 开采辅助活动                             | 2002-02-05 |
| B12      | 其他采矿业                               | 2012-01-01 |
| C13      | 农副食品加工业                           | 1993-12-15 |
| C14      | 食品制造业                               | 1994-08-18 |
| C15      | 酒、饮料和精制茶制造业                   | 1992-10-12 |
| C16      | 烟草制品业                               | 2012-01-01 |
| C17      | 纺织业                                   | 1992-06-16 |
| C18      | 纺织服装、服饰业                         | 1993-12-31 |
| C19      | 皮革、毛皮、羽毛及其制品和制鞋业         | 1994-04-04 |
| C20      | 木材加工和木、竹、藤、棕、草制品业       | 2005-05-10 |
| C21      | 家具制造业                               | 1996-04-25 |
| C22      | 造纸和纸制品业                           | 1993-03-12 |
| C23      | 印刷和记录媒介复制业                     | 1994-02-24 |
| C24      | 文教、工美、体育和娱乐用品制造业         | 2007-01-10 |
| C25      | 石油加工、炼焦和核燃料加工业             | 1993-10-25 |
| C26      | 化学原料和化学制品制造业                 | 1990-12-19 |
| C27      | 医药制造业                               | 1993-06-29 |
| C28      | 化学纤维制造业                           | 1993-07-28 |
| C29      | 橡胶和塑料制品业                         | 1992-08-28 |
| C30      | 非金属矿物制品业                         | 1992-02-28 |
| C31      | 黑色金属冶炼和压延加工业                 | 1994-01-06 |
| C32      | 有色金属冶炼和压延加工业                 | 1996-02-15 |
| C33      | 金属制品业                               | 1993-11-30 |
| C34      | 通用设备制造业                           | 1992-03-27 |
| C35      | 专用设备制造业                           | 1992-07-01 |
| C36      | 汽车制造业                               | 1992-07-24 |
| C37      | 铁路、船舶、航空航天和其他运输设备制造业 | 1992-03-31 |
| C38      | 电气机械和器材制造业                     | 1990-12-19 |
| C39      | 计算机、通信和其他电子设备制造业         | 1990-12-19 |
| C40      | 仪器仪表制造业                           | 1993-09-17 |
| C41      | 其他制造业                               | 1992-08-14 |
| C42      | 废弃资源综合利用业                       | 2012-10-26 |
| C43      | 金属制品、机械和设备修理业               | 2010-09-29 |
| D44      | 电力、热力生产和供应业                   | 1993-04-16 |
| D45      | 燃气生产和供应业                         | 2000-12-11 |
| D46      | 水的生产和供应业                         | 1994-02-24 |
| E47      | 房屋建筑业                               | 1993-04-29 |
| E48      | 土木工程建筑业                           | 1994-01-28 |
| E49      | 建筑安装业                               | 2009-05-19 |
| E50      | 建筑装饰和其他建筑业                     | 1997-05-22 |
| F51      | 批发业                                   | 1992-05-06 |
| F52      | 零售业                                   | 1992-09-02 |
| G53      | 铁路运输业                               | 1998-05-11 |
| G54      | 道路运输业                               | 1991-01-14 |
| G55      | 水上运输业                               | 1993-11-19 |
| G56      | 航空运输业                               | 1997-11-05 |
| G57      | 管道运输业                               | 1996-11-04 |
| G58      | 装卸搬运和运输代理业                     | 1993-05-05 |
| G59      | 仓储业                                   | 1996-06-14 |
| G60      | 邮政业                                   | 2017-02-16 |
| H61      | 住宿业                                   | 1993-11-18 |
| H62      | 餐饮业                                   | 1997-04-30 |
| I63      | 电信、广播电视和卫星传输服务             | 1992-12-02 |
| I64      | 互联网和相关服务                         | 1992-05-07 |
| I65      | 软件和信息技术服务业                     | 1992-08-20 |
| J66      | 货币金融服务                             | 1991-04-03 |
| J67      | 资本市场服务                             | 1994-01-10 |
| J68      | 保险业                                   | 2007-01-09 |
| J69      | 其他金融业                               | 2012-10-26 |
| K70      | 房地产业                                 | 1992-01-13 |
| L71      | 租赁业                                   | 1997-01-30 |
| L72      | 商务服务业                               | 1996-08-29 |
| M73      | 研究和试验发展                           | 2012-10-26 |
| M74      | 专业技术服务业                           | 2007-02-15 |
| M75      | 科技推广和应用服务业                     | 2009-02-11 |
| N76      | 水利管理业                               | 2012-01-01 |
| N77      | 生态保护和环境治理业                     | 2012-10-26 |
| N78      | 公共设施管理业                           | 1992-08-07 |
| O79      | 居民服务业                               | 1993-06-14 |
| O80      | 机动车、电子产品和日用产品修理业         | 2012-01-01 |
| O81      | 其他服务业                               | 1998-03-27 |
| P82      | 教育                                     | 2012-10-26 |
| Q83      | 卫生                                     | 2007-02-05 |
| Q84      | 社会工作                                 | 2012-01-01 |
| R85      | 新闻和出版业                             | 1992-12-08 |
| R86      | 广播、电视、电影和影视录音制作业         | 1994-02-24 |
| R87      | 文化艺术业                               | 2012-10-26 |
| R88      | 体育                                     | 2017-03-24 |
| R89      | 娱乐业                                   | 2012-01-01 |
| S90      | 综合                                     | 1990-12-10 |

 

## 聚宽行业

### 聚宽一级行业

| 行业代码 | 行业名称   | 起始日期   |
| -------- | ---------- | ---------- |
| HY001    | 能源       | 1999-12-30 |
| HY002    | 材料       | 1999-12-30 |
| HY003    | 工业       | 1999-12-30 |
| HY004    | 可选消费指 | 1999-12-30 |
| HY005    | 日常消费   | 1999-12-30 |
| HY006    | 医疗保健   | 1999-12-30 |
| HY007    | 金融       | 1999-12-30 |
| HY008    | 信息技术   | 1999-12-30 |
| HY009    | 电信服务   | 1999-12-30 |
| HY010    | 公用事业   | 1999-12-30 |
| HY011    | 房地产     | 1999-12-30 |

 

### 聚宽二级行业

| 行业代码 | 行业名称                   | 起始日期   |
| -------- | -------------------------- | ---------- |
| HY401    | 石油天然气设备与服务       | 1998-11-27 |
| HY402    | 综合性石油天然气           | 2001-08-09 |
| HY403    | 石油天然气勘探与生产       | 1996-06-28 |
| HY404    | 石油与天然气的炼制和销售   | 1993-06-07 |
| HY405    | 基础化工                   | 1992-05-06 |
| HY406    | 多元化工                   | 2011-03-11 |
| HY407    | 化肥与农用化工             | 1993-10-29 |
| HY408    | 工业气体                   | 2011-02-21 |
| HY409    | 特种化工                   | 1993-03-17 |
| HY410    | 建材                       | 1992-03-02 |
| HY411    | 金属与玻璃容器             | 1996-12-04 |
| HY412    | 纸包装                     | 1998-04-16 |
| HY413    | 铝                         | 1996-02-16 |
| HY414    | 金属非金属                 | 1993-11-19 |
| HY415    | 黄金                       | 2003-08-15 |
| HY416    | 贵金属与矿石               | 2003-05-19 |
| HY417    | 钢铁                       | 1992-03-27 |
| HY418    | 林木产品                   | 1998-10-08 |
| HY419    | 纸制品                     | 1996-06-26 |
| HY420    | 航天航空与国防             | 1997-06-27 |
| HY421    | 建筑产品                   | 1995-11-30 |
| HY422    | 建筑与工程                 | 1994-01-31 |
| HY423    | 电气部件与设备             | 1993-11-26 |
| HY424    | 重型电气设备               | 1995-10-11 |
| HY425    | 综合类行业                 | 1990-12-19 |
| HY426    | 建筑机械与重型卡车         | 1993-10-29 |
| HY427    | 工业机械                   | 1992-03-27 |
| HY428    | 贸易公司与工业品经销商     | 1993-06-21 |
| HY429    | 商业印刷                   | 1993-10-18 |
| HY432    | 环境与设施服务             | 1993-05-18 |
| HY433    | 办公服务与用品             | 1992-07-22 |
| HY434    | 航空货运与物流             | 2000-08-01 |
| HY435    | 航空                       | 1997-06-27 |
| HY436    | 海运                       | 1993-11-19 |
| HY437    | 铁路运输                   | 1998-05-12 |
| HY438    | 公路运输                   | 1991-01-14 |
| HY439    | 机场服务                   | 1996-06-03 |
| HY440    | 公路与铁路                 | 1996-08-16 |
| HY441    | 海港与服务                 | 1993-05-06 |
| HY442    | 机动车零配件与设备         | 1993-06-11 |
| HY443    | 轮胎与橡胶                 | 1992-08-31 |
| HY444    | 汽车制造                   | 1992-07-27 |
| HY445    | 摩托车制造                 | 1993-12-06 |
| HY446    | 消费电子产品               | 1990-12-19 |
| HY447    | 家庭装饰品                 | 1996-04-25 |
| HY448    | 住宅建筑                   | 1997-04-15 |
| HY449    | 家用电器                   | 1993-03-04 |
| HY450    | 家用器具与特殊消费品       | 1993-12-06 |
| HY451    | 休闲用品                   | 1992-04-01 |
| HY452    | 摄影用品                   | 1997-05-29 |
| HY453    | 服装、服饰与奢侈品         | 1992-09-03 |
| HY454    | 鞋类                       | 1999-06-09 |
| HY455    | 纺织品                     | 1992-03-27 |
| HY457    | 酒店、度假村与豪华游轮     | 1993-06-07 |
| HY458    | 休闲设施                   | 2010-12-10 |
| HY459    | 餐馆                       | 1993-05-04 |
| HY460    | 广告                       | 1999-03-26 |
| HY461    | 有线和卫星电视             | 1997-10-31 |
| HY462    | 电影与娱乐                 | 1997-06-17 |
| HY463    | 出版                       | 2002-03-01 |
| HY464    | 消费品经销商               | 1993-01-08 |
| HY465    | 售货目录零售               | 2015-01-22 |
| HY466    | 互联网零售                 | 2015-06-12 |
| HY467    | 百货商店                   | 1992-11-23 |
| HY468    | 综合货品商店               | 2012-11-27 |
| HY469    | 服装零售                   | 1994-02-04 |
| HY470    | 电脑与电子产品零售         | 1994-02-24 |
| HY471    | 家庭装潢零售               | 2003-05-01 |
| HY472    | 专卖店                     | 1997-06-26 |
| HY473    | 药品零售                   | 1994-03-25 |
| HY474    | 食品分销商                 | 1997-01-13 |
| HY476    | 大卖场与超市               | 1994-05-23 |
| HY477    | 啤酒                       | 1993-08-30 |
| HY478    | 白酒与葡萄酒               | 1994-01-07 |
| HY479    | 软饮料                     | 1992-10-13 |
| HY480    | 农产品                     | 1993-04-30 |
| HY481    | 食品加工与肉类             | 1993-11-01 |
| HY483    | 家庭用品                   | 1990-12-19 |
| HY484    | 个人用品                   | 2001-03-16 |
| HY485    | 医疗保健设备               | 1996-08-23 |
| HY486    | 医疗保健用品               | 1996-07-19 |
| HY487    | 保健护理产品经销商         | 2001-02-07 |
| HY488    | 保健护理服务               | 2011-07-20 |
| HY489    | 保健护理设施               | 2007-04-28 |
| HY491    | 生物科技                   | 1994-05-30 |
| HY492    | 西药                       | 1993-06-30 |
| HY493    | 多元化银行                 | 1991-04-04 |
| HY494    | 区域性银行                 | 2007-07-20 |
| HY496    | 其他多元金融服务           | 2015-01-05 |
| HY497    | 多领域控股                 | 1993-04-26 |
| HY500    | 资产管理与托管银行         | 1994-01-31 |
| HY501    | 投资银行业与经纪业         | 1999-12-30 |
| HY504    | 人寿与健康保险             | 2007-01-10 |
| HY505    | 多元化保险                 | 2007-03-02 |
| HY506    | 财产与意外伤害保险         | 2014-05-05 |
| HY509    | 房地产开发                 | 1990-12-19 |
| HY510    | 互联网软件与服务           | 2006-12-18 |
| HY511    | 信息科技咨询与其它服务     | 1993-05-04 |
| HY512    | 数据处理与外包服务         | 2000-08-19 |
| HY513    | 应用软件                   | 1996-06-19 |
| HY514    | 系统软件                   | 2015-12-11 |
| HY515    | 家庭娱乐软件               | 2012-05-14 |
| HY516    | 通信设备                   | 1993-10-19 |
| HY517    | 电脑硬件                   | 1996-08-08 |
| HY518    | 电脑存储与外围设备         | 1992-06-02 |
| HY519    | 电子元件                   | 1990-12-19 |
| HY520    | 电子制造服务               | 2007-06-14 |
| HY521    | 技术产品经销商             | 1993-06-14 |
| HY523    | 半导体设备                 | 2009-12-28 |
| HY524    | 半导体产品                 | 1997-05-20 |
| HY525    | 非传统电信运营商           | 2009-03-27 |
| HY526    | 综合电信服务               | 2002-10-10 |
| HY527    | 无线电信业务               | 1993-04-07 |
| HY528    | 电力                       | 1993-04-16 |
| HY529    | 燃气                       | 2000-12-11 |
| HY530    | 复合型公用事业             | 1994-08-10 |
| HY531    | 水务                       | 1993-05-18 |
| HY570    | 化纤                       | 1992-10-13 |
| HY571    | 煤炭与消费用燃料           | 1996-08-09 |
| HY572    | 中药                       | 1993-12-16 |
| HY573    | 石油与天然气的储存和运输   | 1997-02-27 |
| HY574    | 教育服务                   | 2010-04-11 |
| HY576    | 汽车零售                   | 1999-08-21 |
| HY578    | 医疗保健技术               | 2015-01-13 |
| HY579    | 生命科学工具和服务         | 2012-08-20 |
| HY587    | 独立电力生产商与能源贸易商 | 2012-07-11 |
| HY588    | 综合支持服务               | 2011-08-30 |
| HY591    | 调查和咨询服务             | 2009-11-02 |
| HY593    | 房地产经营公司             | 2009-08-18 |
| HY595    | 房地产服务                 | 2009-08-31 |
| HY596    | 电子设备和仪器             | 1990-12-20 |
| HY597    | 白银                       | 2014-01-29 |
| HY598    | 农用农业机械               | 1994-07-04 |
| HY601    | 新能源发电业者             | 1998-04-03 |

 

## 申万行业

### 申万一级行业

| 行业代码 | 行业名称                        | 起始日期   |
| -------- | ------------------------------- | ---------- |
| 801010   | 农林牧渔I                       | 2004-02-09 |
| 801020   | 采掘I                           | 2004-02-10 |
| 801030   | 化工I                           | 2004-02-10 |
| 801040   | 钢铁I                           | 2004-02-10 |
| 801050   | 有色金属I                       | 2004-02-10 |
| 801060   | 建筑建材I(于2014年02月21日废弃) | 2004-02-10 |
| 801070   | 机械设备I(于2014年02月21日废弃) | 2004-02-10 |
| 801080   | 电子I                           | 2004-02-10 |
| 801090   | 交运设备I(于2014年02月21日废弃) | 2004-02-10 |
| 801100   | 信息设备I(于2014年02月21日废弃) | 2004-02-10 |
| 801110   | 家用电器I                       | 2004-02-10 |
| 801120   | 食品饮料I                       | 2004-02-10 |
| 801130   | 纺织服装I                       | 2004-02-10 |
| 801140   | 轻工制造I                       | 2004-02-10 |
| 801150   | 医药生物I                       | 2004-02-10 |
| 801160   | 公用事业I                       | 2004-02-10 |
| 801170   | 交通运输I                       | 2004-02-10 |
| 801180   | 房地产I                         | 2004-02-10 |
| 801190   | 金融服务I(于2014年02月21日废弃) | 2004-02-10 |
| 801200   | 商业贸易I                       | 2004-02-10 |
| 801210   | 休闲服务I                       | 2004-02-10 |
| 801220   | 信息服务I(于2014年02月21日废弃) | 2004-02-10 |
| 801230   | 综合I                           | 2004-02-10 |
| 801710   | 建筑材料I                       | 2014-02-21 |
| 801720   | 建筑装饰I                       | 2014-02-21 |
| 801730   | 电气设备I                       | 2014-02-21 |
| 801740   | 国防军工I                       | 2014-02-21 |
| 801750   | 计算机I                         | 2014-02-21 |
| 801760   | 传媒I                           | 2014-02-21 |
| 801770   | 通信I                           | 2014-02-21 |
| 801780   | 银行I                           | 2014-02-21 |
| 801790   | 非银金融I                       | 2014-02-21 |
| 801880   | 汽车I                           | 2014-02-21 |
| 801890   | 机械设备I                       | 2014-02-21 |

 

### 申万二级行业

| 行业代码 | 行业名称                               | 起始日期   | 父级行业代码 |
| -------- | -------------------------------------- | ---------- | ------------ |
| 801011   | 林业II                                 | 2005-05-26 | 801010       |
| 801012   | 农产品加工II                           | 2005-05-26 | 801010       |
| 801013   | 农业综合II                             | 2005-05-26 | 801010       |
| 801014   | 饲料II                                 | 2005-05-26 | 801010       |
| 801015   | 渔业II                                 | 2005-05-26 | 801010       |
| 801016   | 种植业II                               | 2005-05-26 | 801010       |
| 801017   | 畜禽养殖II                             | 2011-09-23 | 801010       |
| 801018   | 动物保健II                             | 2011-09-23 | 801010       |
| 801021   | 煤炭开采II                             | 2005-05-26 | 801020       |
| 801022   | 其他采掘II                             | 2005-05-26 | 801020       |
| 801023   | 石油开采II                             | 2005-05-26 | 801020       |
| 801024   | 采掘服务II                             | 2011-09-23 | 801020       |
| 801031   | 化工新材料II(于2014年02月21日废弃)     | 2005-05-26 | 801030       |
| 801032   | 化学纤维II                             | 2005-05-26 | 801030       |
| 801033   | 化学原料II                             | 2005-05-26 | 801030       |
| 801034   | 化学制品II                             | 2005-05-26 | 801030       |
| 801035   | 石油化工II                             | 2005-05-26 | 801030       |
| 801036   | 塑料II                                 | 2005-05-27 | 801030       |
| 801037   | 橡胶II                                 | 2005-05-27 | 801030       |
| 801041   | 钢铁II                                 | 2005-05-27 | 801040       |
| 801042   | 金属制品II(于2008年06月02日废弃)       | 2005-05-27 | 801040       |
| 801051   | 金属非金属新材料II                     | 2005-05-27 | 801050       |
| 801052   | 有色金属冶炼II(于2014年02月21日废弃)   | 2005-05-27 | 801050       |
| 801053   | 黄金II                                 | 2014-02-21 | 801050       |
| 801054   | 稀有金属II                             | 2014-02-21 | 801050       |
| 801055   | 工业金属II                             | 2014-02-21 | 801050       |
| 801061   | 建筑材料II(于2014年02月21日废弃)       | 2005-05-27 | 801710       |
| 801062   | 建筑装饰II(于2014年02月21日废弃)       | 2005-05-27 | 801720       |
| 801071   | 电气设备II(于2014年02月21日废弃)       | 2005-05-27 | 801730       |
| 801072   | 通用机械II                             | 2005-05-27 | 801890       |
| 801073   | 仪器仪表II                             | 2005-05-27 | 801890       |
| 801074   | 专用设备II                             | 2005-05-27 | 801890       |
| 801075   | 金属制品II                             | 2008-06-26 | 801890       |
| 801076   | 运输设备II                             | 2014-02-21 | 801890       |
| 801081   | 半导体II                               | 2005-05-27 | 801080       |
| 801082   | 其他电子II                             | 2005-05-27 | 801080       |
| 801083   | 元件II                                 | 2005-05-27 | 801080       |
| 801084   | 光学光电子II                           | 2005-05-27 | 801080       |
| 801085   | 电子制造II                             | 2011-09-23 | 801080       |
| 801091   | 非汽车交运设备II(于2014年02月21日废弃) | 2005-05-27 | 801090       |
| 801092   | 汽车服务II                             | 2005-05-27 | 801880       |
| 801093   | 汽车零部件II                           | 2005-05-27 | 801880       |
| 801094   | 汽车整车II                             | 2005-05-27 | 801880       |
| 801101   | 计算机设备II                           | 2005-05-27 | 801750       |
| 801102   | 通信设备II                             | 2005-05-27 | 801770       |
| 801111   | 白色家电II                             | 2005-05-27 | 801110       |
| 801112   | 视听器材II                             | 2005-05-27 | 801110       |
| 801121   | 食品加工II(于2011年10月10日废弃)       | 2005-05-27 | 801120       |
| 801122   | 食品制造II(于2011年10月10日废弃)       | 2005-05-27 | 801120       |
| 801123   | 饮料制造II                             | 2005-05-27 | 801120       |
| 801124   | 食品加工II                             | 2011-09-23 | 801120       |
| 801131   | 纺织制造II                             | 2005-05-27 | 801130       |
| 801132   | 服装家纺II                             | 2005-05-27 | 801130       |
| 801141   | 包装印刷II                             | 2005-05-27 | 801140       |
| 801142   | 家用轻工II                             | 2005-05-27 | 801140       |
| 801143   | 造纸II                                 | 2005-05-27 | 801140       |
| 801144   | 其他轻工制造II                         | 2014-02-21 | 801140       |
| 801151   | 化学制药II                             | 2005-05-27 | 801150       |
| 801152   | 生物制品II                             | 2005-05-27 | 801150       |
| 801153   | 医疗器械II                             | 2005-05-27 | 801150       |
| 801154   | 医药商业II                             | 2005-05-27 | 801150       |
| 801155   | 中药II                                 | 2005-05-27 | 801150       |
| 801156   | 医疗服务II                             | 2007-07-04 | 801150       |
| 801161   | 电力II                                 | 2005-05-27 | 801160       |
| 801162   | 环保工程及服务II                       | 2005-05-27 | 801160       |
| 801163   | 燃气II                                 | 2005-05-27 | 801160       |
| 801164   | 水务II                                 | 2005-05-27 | 801160       |
| 801171   | 港口II                                 | 2005-05-27 | 801170       |
| 801172   | 公交II                                 | 2005-05-27 | 801170       |
| 801173   | 航空运输II                             | 2005-05-27 | 801170       |
| 801174   | 机场II                                 | 2005-05-27 | 801170       |
| 801175   | 高速公路II                             | 2005-05-27 | 801170       |
| 801176   | 航运II                                 | 2005-05-27 | 801170       |
| 801177   | 铁路运输II                             | 2005-05-27 | 801170       |
| 801178   | 物流II                                 | 2005-05-27 | 801170       |
| 801181   | 房地产开发II                           | 2005-05-27 | 801180       |
| 801182   | 园区开发II                             | 2005-05-27 | 801180       |
| 801191   | 多元金融II                             | 2005-05-27 | 801790       |
| 801192   | 银行II                                 | 2005-05-27 | 801780       |
| 801193   | 证券II                                 | 2005-05-27 | 801790       |
| 801194   | 保险II                                 | 2007-01-18 | 801790       |
| 801201   | 零售II(于2014年02月21日废弃)           | 2005-05-27 | 801200       |
| 801202   | 贸易II                                 | 2005-05-27 | 801200       |
| 801203   | 一般零售II                             | 2014-02-21 | 801200       |
| 801204   | 专业零售II                             | 2014-02-21 | 801200       |
| 801205   | 商业物业经营II                         | 2014-02-21 | 801200       |
| 801211   | 餐饮II                                 | 2005-05-27 | 801210       |
| 801212   | 景点II                                 | 2005-05-27 | 801210       |
| 801213   | 酒店II                                 | 2005-05-27 | 801210       |
| 801214   | 旅游综合II                             | 2005-05-27 | 801210       |
| 801215   | 其他休闲服务II                         | 2014-02-21 | 801210       |
| 801221   | 传媒II(于2014年02月21日废弃)           | 2005-05-27 | 801760       |
| 801222   | 计算机应用II                           | 2005-05-27 | 801750       |
| 801223   | 通信运营II                             | 2005-05-27 | 801770       |
| 801224   | 网络服务II(于2014年02月21日废弃)       | 2005-05-27 | 801220       |
| 801231   | 综合II                                 | 2005-05-27 | 801230       |
| 801711   | 水泥制造II                             | 2014-02-21 | 801710       |
| 801712   | 玻璃制造II                             | 2014-02-21 | 801710       |
| 801713   | 其他建材II                             | 2014-02-21 | 801710       |
| 801721   | 房屋建设II                             | 2014-02-21 | 801720       |
| 801722   | 装修装饰II                             | 2014-02-21 | 801720       |
| 801723   | 基础建设II                             | 2014-02-21 | 801720       |
| 801724   | 专业工程II                             | 2014-02-21 | 801720       |
| 801725   | 园林工程II                             | 2014-02-21 | 801720       |
| 801731   | 电机II                                 | 2014-02-21 | 801730       |
| 801732   | 电气自动化设备II                       | 2014-02-21 | 801730       |
| 801733   | 电源设备II                             | 2014-02-21 | 801730       |
| 801734   | 高低压设备II                           | 2014-02-21 | 801730       |
| 801741   | 航天装备II                             | 2014-02-21 | 801740       |
| 801742   | 航空装备II                             | 2014-02-21 | 801740       |
| 801743   | 地面兵装II                             | 2014-02-21 | 801740       |
| 801744   | 船舶制造II                             | 2014-02-21 | 801740       |
| 801751   | 营销传播II                             | 2014-02-21 | 801760       |
| 801752   | 互联网传媒II                           | 2014-02-21 | 801760       |
| 801761   | 文化传媒II                             | 2014-02-21 | 801760       |
| 801881   | 其他交运设备II                         | 2014-02-21 | 801880       |

 

### 申万三级行业

 

| 行业代码 | 行业名称                                  | 起始日期   | 父级行业代码 |
| -------- | ----------------------------------------- | ---------- | ------------ |
| 850111   | 种子生产III                               | 2005-05-27 | 801016       |
| 850112   | 粮食种植III                               | 2005-05-27 | 801016       |
| 850113   | 其他种植业III                             | 2005-05-27 | 801016       |
| 850121   | 海洋捕捞III                               | 2005-05-27 | 801015       |
| 850122   | 水产养殖III                               | 2005-05-27 | 801015       |
| 850131   | 林业III                                   | 2005-05-27 | 801011       |
| 850141   | 饲料III                                   | 2005-05-27 | 801014       |
| 850151   | 果蔬加工III                               | 2005-05-27 | 801012       |
| 850152   | 粮油加工III                               | 2005-05-27 | 801012       |
| 850153   | 畜禽加工III(于2011年10月10日废弃)         | 2005-05-27 | 801012       |
| 850154   | 其他农产品加工III                         | 2011-09-23 | 801012       |
| 850161   | 农业综合III                               | 2005-05-27 | 801013       |
| 850171   | 畜禽养殖III                               | 2011-09-23 | 801017       |
| 850181   | 动物保健III                               | 2011-09-23 | 801018       |
| 850211   | 石油开采III                               | 2005-05-27 | 801023       |
| 850221   | 煤炭开采III                               | 2005-05-27 | 801021       |
| 850222   | 焦炭加工III                               | 2005-05-27 | 801021       |
| 850231   | 其他采掘III                               | 2005-05-27 | 801022       |
| 850241   | 油气钻采服务III                           | 2011-09-23 | 801024       |
| 850242   | 其他采掘服务III                           | 2011-09-23 | 801024       |
| 850311   | 石油加工III                               | 2005-05-27 | 801035       |
| 850313   | 石油贸易III                               | 2005-05-27 | 801035       |
| 850321   | 纯碱III                                   | 2005-05-27 | 801033       |
| 850322   | 氯碱III                                   | 2005-05-27 | 801033       |
| 850323   | 无机盐III                                 | 2005-05-27 | 801033       |
| 850324   | 其他化学原料III                           | 2007-07-04 | 801033       |
| 850331   | 氮肥III                                   | 2005-05-27 | 801034       |
| 850332   | 磷肥III                                   | 2005-05-27 | 801034       |
| 850333   | 农药III                                   | 2005-05-27 | 801034       |
| 850334   | 日用化学产品III                           | 2005-05-27 | 801034       |
| 850335   | 涂料油漆油墨制造III                       | 2005-05-27 | 801034       |
| 850336   | 钾肥III                                   | 2005-05-27 | 801034       |
| 850337   | 民爆用品III                               | 2008-06-26 | 801034       |
| 850338   | 纺织化学用品III                           | 2008-06-26 | 801034       |
| 850339   | 其他化学制品III                           | 2008-06-26 | 801034       |
| 850341   | 涤纶III                                   | 2005-05-27 | 801032       |
| 850342   | 维纶III                                   | 2005-05-27 | 801032       |
| 850343   | 粘胶III                                   | 2005-05-27 | 801032       |
| 850344   | 其他纤维III                               | 2005-05-27 | 801032       |
| 850345   | 氨纶III                                   | 2008-06-27 | 801032       |
| 850351   | 其他塑料制品III                           | 2005-05-27 | 801036       |
| 850352   | 合成革III                                 | 2011-09-23 | 801036       |
| 850353   | 改性塑料III                               | 2011-09-23 | 801036       |
| 850361   | 轮胎III                                   | 2005-05-27 | 801037       |
| 850362   | 其他橡胶制品III                           | 2008-06-27 | 801037       |
| 850363   | 炭黑III                                   | 2011-09-23 | 801037       |
| 850371   | 其他化工新材料III(于2014年02月21日废弃)   | 2005-05-27 | 801031       |
| 850372   | 聚氨酯III                                 | 2011-09-23 | 801034       |
| 850373   | 玻纤III                                   | 2011-09-23 | 801034       |
| 850381   | 复合肥III                                 | 2011-09-23 | 801034       |
| 850382   | 氟化工及制冷剂III                         | 2011-09-23 | 801034       |
| 850383   | 磷化工及磷酸盐III                         | 2011-09-23 | 801034       |
| 850411   | 普钢III                                   | 2005-05-27 | 801041       |
| 850412   | 特钢III                                   | 2005-05-27 | 801041       |
| 850421   | 金属制品III(于2008年06月02日废弃)         | 2005-05-27 | 801042       |
| 850511   | 有色金属冶炼III(于2007年07月02日废弃)     | 2005-05-27 | 801052       |
| 850512   | 铝III(于2014年02月21日废弃)               | 2007-07-04 | 801052       |
| 850513   | 铜III(于2014年02月21日废弃)               | 2007-07-04 | 801052       |
| 850514   | 铅锌III(于2014年02月21日废弃)             | 2007-07-04 | 801052       |
| 850515   | 黄金III(于2014年02月21日废弃)             | 2007-07-04 | 801052       |
| 850516   | 小金属III(于2014年02月21日废弃)           | 2007-07-04 | 801052       |
| 850521   | 金属新材料III                             | 2005-05-27 | 801051       |
| 850522   | 磁性材料III                               | 2008-06-26 | 801051       |
| 850523   | 非金属新材料III                           | 2008-06-26 | 801051       |
| 850531   | 黄金III                                   | 2014-02-21 | 801053       |
| 850541   | 稀土III                                   | 2014-02-21 | 801054       |
| 850542   | 钨III                                     | 2014-02-21 | 801054       |
| 850543   | 锂III                                     | 2014-02-21 | 801054       |
| 850544   | 其他稀有小金属III                         | 2014-02-21 | 801054       |
| 850551   | 铝III                                     | 2014-02-21 | 801055       |
| 850552   | 铜III                                     | 2014-02-21 | 801055       |
| 850553   | 铅锌III                                   | 2014-02-21 | 801055       |
| 850611   | 玻璃制造III                               | 2005-05-27 | 801712       |
| 850612   | 水泥制造III                               | 2005-05-27 | 801711       |
| 850613   | 陶瓷制造III(于2014年02月21日废弃)         | 2005-05-27 | 801061       |
| 850614   | 其他建材III                               | 2005-05-27 | 801713       |
| 850615   | 耐火材料III                               | 2011-09-23 | 801713       |
| 850616   | 管材III                                   | 2011-09-23 | 801713       |
| 850621   | 建筑施工III(于2011年10月10日废弃)         | 2005-05-27 | 801062       |
| 850622   | 装饰园林III(于2014年02月21日废弃)         | 2005-05-27 | 801062       |
| 850623   | 房屋建设III                               | 2011-09-23 | 801721       |
| 850624   | 基础建设III(于2014年02月21日废弃)         | 2011-09-23 | 801062       |
| 850625   | 专业工程III(于2014年02月21日废弃)         | 2011-09-23 | 801062       |
| 850711   | 机床工具III                               | 2005-05-27 | 801072       |
| 850712   | 机械基础件III                             | 2005-05-27 | 801072       |
| 850713   | 磨具磨料III                               | 2005-05-27 | 801072       |
| 850714   | 内燃机III                                 | 2005-05-27 | 801072       |
| 850715   | 制冷空调设备III                           | 2005-05-27 | 801072       |
| 850716   | 其它通用机械III                           | 2005-05-27 | 801072       |
| 850721   | 纺织服装设备III                           | 2005-05-27 | 801074       |
| 850722   | 工程机械III                               | 2005-05-27 | 801074       |
| 850723   | 农用机械III                               | 2005-05-27 | 801074       |
| 850724   | 重型机械III                               | 2005-05-27 | 801074       |
| 850725   | 冶金矿采化工设备III                       | 2005-05-27 | 801074       |
| 850726   | 印刷包装机械III                           | 2005-05-27 | 801074       |
| 850727   | 其它专用机械III                           | 2005-05-27 | 801074       |
| 850728   | 楼宇设备III                               | 2011-09-23 | 801074       |
| 850729   | 环保设备III                               | 2011-09-23 | 801074       |
| 850731   | 仪器仪表III                               | 2005-05-27 | 801073       |
| 850741   | 电机III                                   | 2005-05-27 | 801731       |
| 850742   | 电气自控设备III(于2014年02月21日废弃)     | 2005-05-27 | 801071       |
| 850743   | 电源设备III(于2014年02月21日废弃)         | 2005-05-27 | 801071       |
| 850744   | 输变电设备III(于2014年02月21日废弃)       | 2005-05-27 | 801071       |
| 850745   | 其他电力设备III(于2014年02月21日废弃)     | 2005-05-27 | 801071       |
| 850751   | 金属制品III                               | 2008-06-26 | 801075       |
| 850811   | 集成电路III                               | 2005-05-27 | 801081       |
| 850812   | 分立器件III                               | 2007-07-04 | 801081       |
| 850813   | 半导体材料III                             | 2007-07-04 | 801081       |
| 850821   | 元件III(于2011年10月10日废弃)             | 2005-05-27 | 801083       |
| 850822   | 印制电路板III                             | 2011-09-23 | 801083       |
| 850823   | 被动元件III                               | 2011-09-23 | 801083       |
| 850831   | 显示器件III                               | 2005-05-27 | 801084       |
| 850832   | LEDIII                                    | 2011-09-23 | 801084       |
| 850833   | 光学元件III                               | 2011-09-23 | 801084       |
| 850841   | 其他电子III                               | 2005-05-27 | 801082       |
| 850851   | 电子系统组装III                           | 2011-09-23 | 801085       |
| 850852   | 电子零部件制造III                         | 2011-09-23 | 801085       |
| 850911   | 乘用车III                                 | 2005-05-27 | 801094       |
| 850912   | 商用载货车III                             | 2005-05-27 | 801094       |
| 850913   | 商用载客车III                             | 2005-05-27 | 801094       |
| 850914   | 专用汽车III(于2014年02月21日废弃)         | 2005-05-27 | 801094       |
| 850921   | 汽车零部件III                             | 2005-05-27 | 801093       |
| 850931   | 摩托车III(于2014年02月21日废弃)           | 2005-05-27 | 801091       |
| 850932   | 农机设备III(于2011年10月10日废弃)         | 2005-05-27 | 801091       |
| 850933   | 其他交运设备III(于2007年07月02日废弃)     | 2005-05-27 | 801091       |
| 850934   | 航空航天设备III(于2014年02月21日废弃)     | 2007-07-04 | 801091       |
| 850935   | 船舶制造III                               | 2007-07-04 | 801744       |
| 850936   | 铁路设备III                               | 2007-07-04 | 801076       |
| 850937   | 其他交运设备III(于2014年02月21日废弃)     | 2011-09-23 | 801091       |
| 850941   | 汽车服务III                               | 2005-05-27 | 801092       |
| 850942   | 其他交运设备服务III(于2014年02月21日废弃) | 2011-09-23 | 801092       |
| 851011   | 交换设备III(于2011年10月10日废弃)         | 2005-05-27 | 801102       |
| 851012   | 终端设备III                               | 2005-05-27 | 801102       |
| 851013   | 通信传输设备III                           | 2005-05-27 | 801102       |
| 851014   | 通信配套服务III                           | 2008-06-26 | 801102       |
| 851021   | 计算机设备III                             | 2005-05-27 | 801101       |
| 851111   | 冰箱III                                   | 2005-05-27 | 801111       |
| 851112   | 空调III                                   | 2005-05-27 | 801111       |
| 851113   | 洗衣机III                                 | 2005-05-27 | 801111       |
| 851114   | 小家电III                                 | 2005-05-27 | 801111       |
| 851115   | 家电零部件III                             | 2005-05-27 | 801111       |
| 851121   | 彩电III                                   | 2005-05-27 | 801112       |
| 851122   | 其它视听器材III                           | 2005-05-27 | 801112       |
| 851211   | 肉制品III(于2011年10月10日废弃)           | 2005-05-27 | 801121       |
| 851212   | 制糖III(于2011年10月10日废弃)             | 2005-05-27 | 801121       |
| 851213   | 食品综合III(于2011年10月10日废弃)         | 2007-07-04 | 801121       |
| 851221   | 调味发酵品III(于2011年10月10日废弃)       | 2005-05-27 | 801122       |
| 851222   | 乳品III(于2011年10月10日废弃)             | 2005-05-27 | 801122       |
| 851231   | 白酒III                                   | 2005-05-27 | 801123       |
| 851232   | 啤酒III                                   | 2005-05-27 | 801123       |
| 851233   | 其他酒类III                               | 2005-05-27 | 801123       |
| 851234   | 软饮料III                                 | 2005-05-27 | 801123       |
| 851235   | 葡萄酒III                                 | 2007-07-04 | 801123       |
| 851236   | 黄酒III                                   | 2008-06-26 | 801123       |
| 851241   | 肉制品III                                 | 2011-09-23 | 801121       |
| 851242   | 调味发酵品III                             | 2011-09-23 | 801121       |
| 851243   | 乳品III                                   | 2011-09-23 | 801121       |
| 851244   | 食品综合III                               | 2011-09-23 | 801121       |
| 851311   | 毛纺III                                   | 2005-05-27 | 801131       |
| 851312   | 棉纺III                                   | 2005-05-27 | 801131       |
| 851313   | 丝绸III                                   | 2005-05-27 | 801131       |
| 851314   | 印染III                                   | 2005-05-27 | 801131       |
| 851315   | 辅料III                                   | 2011-09-23 | 801131       |
| 851316   | 其他纺织III                               | 2011-09-23 | 801131       |
| 851321   | 服装III(于2011年10月10日废弃)             | 2005-05-27 | 801132       |
| 851322   | 男装III                                   | 2011-09-23 | 801132       |
| 851323   | 女装III                                   | 2011-09-23 | 801132       |
| 851324   | 休闲服装III                               | 2011-09-23 | 801132       |
| 851325   | 鞋帽III                                   | 2011-09-23 | 801132       |
| 851326   | 家纺III                                   | 2011-09-23 | 801132       |
| 851327   | 其他服装III                               | 2011-09-23 | 801132       |
| 851411   | 造纸III                                   | 2005-05-27 | 801143       |
| 851421   | 包装印刷III                               | 2005-05-27 | 801141       |
| 851431   | 其他轻工制造III(于2011年10月10日废弃)     | 2005-05-27 | 801142       |
| 851432   | 家具III                                   | 2007-07-04 | 801142       |
| 851433   | 其他家用轻工III                           | 2007-07-04 | 801142       |
| 851434   | 珠宝首饰III                               | 2011-09-23 | 801142       |
| 851435   | 文娱用品III                               | 2011-09-23 | 801142       |
| 851441   | 其他轻工制造III                           | 2014-02-21 | 801144       |
| 851511   | 化学原料药III                             | 2005-05-27 | 801151       |
| 851512   | 化学制剂III                               | 2005-05-27 | 801151       |
| 851513   | 化学药III(于2008年06月02日废弃)           | 2005-05-27 | 801151       |
| 851521   | 中药III                                   | 2005-05-27 | 801155       |
| 851531   | 生物制品III                               | 2005-05-27 | 801152       |
| 851541   | 医药商业III                               | 2005-05-27 | 801154       |
| 851551   | 医疗器械III                               | 2005-05-27 | 801153       |
| 851561   | 医疗服务III                               | 2007-07-04 | 801156       |
| 851611   | 火电III                                   | 2005-05-27 | 801161       |
| 851612   | 水电III                                   | 2005-05-27 | 801161       |
| 851613   | 燃机发电III                               | 2005-05-27 | 801161       |
| 851614   | 热电III                                   | 2005-05-27 | 801161       |
| 851615   | 新能源发电III                             | 2014-02-21 | 801161       |
| 851621   | 水务III                                   | 2005-05-27 | 801164       |
| 851631   | 燃气III                                   | 2005-05-27 | 801163       |
| 851641   | 环保工程及服务III                         | 2005-05-27 | 801162       |
| 851711   | 港口III                                   | 2005-05-27 | 801171       |
| 851721   | 公交III                                   | 2005-05-27 | 801172       |
| 851731   | 高速公路III                               | 2005-05-27 | 801175       |
| 851741   | 航空运输III                               | 2005-05-27 | 801173       |
| 851751   | 机场III                                   | 2005-05-27 | 801174       |
| 851761   | 航运III                                   | 2005-05-27 | 801176       |
| 851771   | 铁路运输III                               | 2005-05-27 | 801177       |
| 851781   | 物流III                                   | 2005-05-27 | 801178       |
| 851811   | 房地产开发III                             | 2005-05-27 | 801181       |
| 851821   | 园区开发III                               | 2005-05-27 | 801182       |
| 851911   | 银行III                                   | 2005-05-27 | 801192       |
| 851921   | 多元金融III                               | 2005-05-27 | 801191       |
| 851931   | 证券III                                   | 2005-05-27 | 801193       |
| 851941   | 保险III                                   | 2007-01-18 | 801194       |
| 852011   | 百货零售III(于2014年02月21日废弃)         | 2005-05-27 | 801201       |
| 852012   | 专业连锁III(于2014年02月21日废弃)         | 2005-05-27 | 801201       |
| 852013   | 商业物业经营III(于2014年02月21日废弃)     | 2007-07-04 | 801201       |
| 852021   | 贸易III                                   | 2005-05-27 | 801202       |
| 852031   | 百货III                                   | 2014-02-21 | 801203       |
| 852032   | 超市III                                   | 2014-02-21 | 801203       |
| 852033   | 多业态零售III                             | 2014-02-21 | 801203       |
| 852041   | 专业连锁III                               | 2014-02-21 | 801204       |
| 852051   | 一般物业经营III                           | 2014-02-21 | 801205       |
| 852052   | 专业市场III                               | 2014-02-21 | 801205       |
| 852111   | 人工景点III                               | 2005-05-27 | 801212       |
| 852112   | 自然景点III                               | 2005-05-27 | 801212       |
| 852121   | 酒店III                                   | 2005-05-27 | 801213       |
| 852131   | 旅游综合III                               | 2005-05-27 | 801214       |
| 852141   | 餐饮III                                   | 2005-05-27 | 801211       |
| 852151   | 其他休闲服务III                           | 2014-02-21 | 801215       |
| 852211   | 通信运营III                               | 2005-05-27 | 801223       |
| 852221   | 互联网信息服务III                         | 2005-05-27 | 801752       |
| 852222   | 移动互联网服务III                         | 2005-05-27 | 801752       |
| 852223   | 其他互联网服务III                         | 2005-05-27 | 801752       |
| 852224   | 有线电视网络III                           | 2005-05-27 | 801761       |
| 852225   | 软件开发III                               | 2014-02-21 | 801222       |
| 852226   | IT服务III                                 | 2014-02-21 | 801222       |
| 852231   | 计算机应用III(于2008年06月02日废弃)       | 2005-05-27 | 801222       |
| 852232   | 软件开发及服务III(于2014年02月21日废弃)   | 2008-06-26 | 801222       |
| 852233   | 系统集成III(于2014年02月21日废弃)         | 2008-06-26 | 801222       |
| 852241   | 平面媒体III                               | 2005-05-27 | 801761       |
| 852242   | 影视动漫III                               | 2005-05-27 | 801761       |
| 852243   | 营销服务III                               | 2011-09-23 | 801751       |
| 852244   | 其他文化传媒III                           | 2011-09-23 | 801761       |
| 852311   | 综合III                                   | 2005-05-27 | 801231       |
| 857221   | 装修装饰III                               | 2014-02-21 | 801722       |
| 857231   | 城轨建设III                               | 2014-02-21 | 801723       |
| 857232   | 路桥施工III                               | 2014-02-21 | 801723       |
| 857233   | 水利工程III                               | 2014-02-21 | 801723       |
| 857234   | 铁路建设III                               | 2014-02-21 | 801723       |
| 857235   | 其他基础建设III                           | 2014-02-21 | 801723       |
| 857241   | 钢结构III                                 | 2014-02-21 | 801724       |
| 857242   | 化学工程III                               | 2014-02-21 | 801724       |
| 857243   | 国际工程承包III                           | 2014-02-21 | 801724       |
| 857244   | 其他专业工程III                           | 2014-02-21 | 801724       |
| 857251   | 园林工程III                               | 2014-02-21 | 801725       |
| 857321   | 电网自动化III                             | 2014-02-21 | 801732       |
| 857322   | 工控自动化III                             | 2014-02-21 | 801732       |
| 857323   | 计量仪表III                               | 2014-02-21 | 801732       |
| 857331   | 综合电力设备商III                         | 2014-02-21 | 801733       |
| 857332   | 风电设备III                               | 2014-02-21 | 801733       |
| 857333   | 光伏设备III                               | 2014-02-21 | 801733       |
| 857334   | 火电设备III                               | 2014-02-21 | 801733       |
| 857335   | 储能设备III                               | 2014-02-21 | 801733       |
| 857336   | 其它电源设备III                           | 2014-02-21 | 801733       |
| 857341   | 高压设备III                               | 2014-02-21 | 801734       |
| 857342   | 中压设备III                               | 2014-02-21 | 801734       |
| 857343   | 低压设备III                               | 2014-02-21 | 801734       |
| 857344   | 线缆部件及其他III                         | 2014-02-21 | 801734       |
| 857411   | 航天装备III                               | 2014-02-21 | 801741       |
| 857421   | 航空装备III                               | 2014-02-21 | 801742       |
| 857431   | 地面兵装III                               | 2014-02-21 | 801743       |
| 858811   | 其他交运设备III                           | 2014-02-21 | 801881       |

# **概念类**


## 概念板块

| 概念代码 | 概念名称           | 起始时间   |
| -------- | ------------------ | ---------- |
| GN102    | 抗艾滋病           | 2011-03-15 |
| GN139    | 土地流转           | 2013-12-31 |
| GN140    | 二胎政策           | 2013-12-31 |
| GN141    | 民营银行           | 2013-12-31 |
| GN142    | 养老产业           | 2013-12-31 |
| GN143    | 尾气治理           | 2013-12-31 |
| GN144    | 网络彩票           | 2013-12-31 |
| GN110    | 安防监控           | 2013-12-31 |
| GN145    | O2O                | 2013-12-31 |
| GN146    | 第三方支付         | 2013-12-31 |
| GN147    | 移动互联网入口     | 2013-12-31 |
| GN138    | 上海自贸区         | 2013-12-31 |
| GN148    | 民营医院           | 2013-12-31 |
| GN150    | 油气管网升级       | 2013-12-31 |
| GN151    | 创投               | 2013-12-31 |
| GN152    | 粤港澳自贸区       | 2013-12-31 |
| GN109    | 石墨烯             | 2013-12-31 |
| GN108    | 涉矿               | 2013-12-31 |
| GN107    | 黄金珠宝           | 2013-12-31 |
| GN106    | 浦东新区           | 2013-12-31 |
| GN105    | 舟山新区           | 2013-12-31 |
| GN113    | 金融改革           | 2013-12-31 |
| GN104    | 聚碳酸酯           | 2013-12-31 |
| GN149    | 油气改革           | 2013-12-31 |
| GN103    | 氟化工             | 2013-12-31 |
| GN001    | 参股金融           | 2013-12-31 |
| GN135    | 电子商务           | 2013-12-31 |
| GN115    | 中日韩自贸区       | 2013-12-31 |
| GN116    | 苹果               | 2013-12-31 |
| GN117    | 三沙               | 2013-12-31 |
| GN118    | 中朝经济特区       | 2013-12-31 |
| GN119    | 3D打印             | 2013-12-31 |
| GN120    | 地热能             | 2013-12-31 |
| GN121    | 美丽中国           | 2013-12-31 |
| GN122    | 太赫兹             | 2013-12-31 |
| GN123    | 智能IC卡           | 2013-12-31 |
| GN124    | PM2.5              | 2013-12-31 |
| GN136    | 互联网金融         | 2013-12-31 |
| GN112    | 宽带提速           | 2013-12-31 |
| GN126    | 机器人             | 2013-12-31 |
| GN127    | 食品安全           | 2013-12-31 |
| GN128    | 4G                 | 2013-12-31 |
| GN129    | 生物疫苗           | 2013-12-31 |
| GN130    | LNG                | 2013-12-31 |
| GN131    | 大数据             | 2013-12-31 |
| GN132    | 智能物流           | 2013-12-31 |
| GN133    | 网络安全           | 2013-12-31 |
| GN134    | 智能穿戴           | 2013-12-31 |
| GN111    | 文化传媒概念       | 2013-12-31 |
| GN125    | 污水处理           | 2013-12-31 |
| GN137    | 智能电视           | 2013-12-31 |
| GN114    | 充电桩             | 2013-12-31 |
| GN087    | 卫星导航           | 2013-12-31 |
| GN079    | 建筑节能           | 2013-12-31 |
| GN080    | 网络游戏           | 2013-12-31 |
| GN081    | 上海本地重组       | 2013-12-31 |
| GN086    | 稀土永磁           | 2013-12-31 |
| GN088    | 通用航空           | 2013-12-31 |
| GN089    | 装饰园林           | 2013-12-31 |
| GN077    | 三网融合           | 2013-12-31 |
| GN090    | 合同能源管理       | 2013-12-31 |
| GN092    | 高端装备制造       | 2013-12-31 |
| GN093    | 智能交通           | 2013-12-31 |
| GN096    | 水利水电建设       | 2013-12-31 |
| GN098    | IPV6               | 2013-12-31 |
| GN099    | 移动互联网         | 2013-12-31 |
| GN100    | 相变存储           | 2013-12-31 |
| GN091    | 云计算             | 2013-12-31 |
| GN076    | 新能源汽车         | 2013-12-31 |
| GN075    | 高铁               | 2013-12-31 |
| GN040    | 垃圾发电           | 2013-12-31 |
| GN028    | 智能电网           | 2013-12-31 |
| GN030    | 物联网             | 2013-12-31 |
| GN031    | 重组               | 2013-12-31 |
| GN032    | 迪士尼             | 2013-12-31 |
| GN034    | 环保概念           | 2013-12-31 |
| GN035    | 新能源             | 2013-12-31 |
| GN036    | 风力发电           | 2013-12-31 |
| GN039    | 锂电池             | 2013-12-31 |
| GN101    | 参股宁德时代       | 2013-12-31 |
| GN041    | 绿色节能照明       | 2013-12-31 |
| GN045    | 太阳能发电         | 2013-12-31 |
| GN046    | 核能核电           | 2013-12-31 |
| GN050    | 海南旅游岛         | 2013-12-31 |
| GN057    | 新材料             | 2013-12-31 |
| GN062    | 生物育种           | 2013-12-31 |
| GN069    | 移动支付           | 2013-12-31 |
| GN074    | 新疆区域振兴       | 2013-12-31 |
| GN153    | 智能家居           | 2014-01-14 |
| GN154    | 超级电容           | 2014-01-24 |
| GN155    | 蓝宝石             | 2014-01-27 |
| GN157    | 智慧医疗           | 2014-01-29 |
| GN156    | 在线教育           | 2014-01-29 |
| GN158    | 移动转售           | 2014-01-30 |
| GN160    | 芯片国产化         | 2014-02-12 |
| GN161    | 传感器             | 2014-02-12 |
| GN162    | 智能汽车           | 2014-02-12 |
| GN159    | 冷链物流           | 2014-02-12 |
| GN163    | 在线旅游           | 2014-02-26 |
| GN164    | 广东国资改革       | 2014-03-03 |
| GN165    | 丝绸之路           | 2014-03-03 |
| GN166    | 燃料电池           | 2014-03-07 |
| GN167    | 京津冀一体化       | 2014-03-19 |
| GN168    | 长江经济带         | 2014-04-30 |
| GN169    | 去IOE              | 2014-05-28 |
| GN170    | 职业教育           | 2014-06-27 |
| GN171    | 国资改革           | 2014-07-16 |
| GN172    | 无人机             | 2014-09-03 |
| GN173    | 赛马               | 2014-10-17 |
| GN174    | 体育               | 2014-10-22 |
| GN175    | 福建自贸区         | 2014-10-29 |
| GN176    | 北部湾自贸区       | 2014-11-07 |
| GN177    | 沪股通50           | 2014-11-13 |
| GN178    | 工业4.0            | 2014-11-27 |
| GN179    | 医药电商           | 2014-12-09 |
| GN181    | 一带一路           | 2015-02-09 |
| GN180    | 天津自贸区         | 2015-02-09 |
| GN182    | 智慧农业           | 2015-03-09 |
| GN183    | 生物识别           | 2015-03-16 |
| GN184    | 基因检测           | 2015-03-17 |
| GN185    | 高送转概念         | 2015-03-27 |
| GN186    | 军民融合           | 2015-04-21 |
| GN187    | 大央企重组         | 2015-04-27 |
| GN188    | 能源互联网         | 2015-05-05 |
| GN189    | 次新股             | 2015-05-18 |
| GN190    | 特高压             | 2015-05-19 |
| GN191    | 跨境电商           | 2015-05-28 |
| GN192    | 员工持股           | 2015-06-08 |
| GN193    | 质押式回购         | 2015-06-11 |
| GN194    | IP流量变现         | 2015-07-27 |
| GN195    | 西藏振兴           | 2015-07-31 |
| GN196    | 国家队             | 2015-08-05 |
| GN197    | ST概念             | 2015-08-12 |
| GN198    | 证金概念           | 2015-08-21 |
| GN199    | 举牌               | 2015-09-23 |
| GN200    | 高校               | 2015-10-15 |
| GN202    | 虚拟现实           | 2015-10-20 |
| GN201    | 人工智能           | 2015-10-20 |
| GN203    | 健康中国           | 2015-10-22 |
| GN204    | 金控平台           | 2015-11-04 |
| GN205    | 海绵城市           | 2015-11-05 |
| GN206    | 汽车后市场         | 2015-11-18 |
| GN207    | 互联网营销         | 2015-12-04 |
| GN208    | 量子通信           | 2015-12-14 |
| GN209    | 动漫               | 2015-12-16 |
| GN210    | 智慧城市           | 2016-02-23 |
| GN211    | 网红经济           | 2016-04-07 |
| GN212    | 征信               | 2016-04-08 |
| GN213    | 债转股             | 2016-04-14 |
| GN214    | OLED               | 2016-05-04 |
| GN215    | 区块链             | 2016-05-22 |
| GN216    | 特斯拉             | 2016-05-23 |
| GN217    | 电子竞技           | 2016-08-01 |
| GN218    | PPP                | 2016-08-30 |
| GN219    | 扶贫               | 2016-09-28 |
| GN220    | 股权转让           | 2016-10-20 |
| GN222    | 共享单车           | 2016-12-26 |
| GN221    | 国企混改           | 2016-12-26 |
| GN1090   | 预增               | 2017-01-03 |
| GN223    | 小程序             | 2017-01-10 |
| GN224    | 5G                 | 2017-02-07 |
| GN225    | 禽流感             | 2017-02-16 |
| GN226    | 共享汽车           | 2017-02-21 |
| GN227    | 新零售             | 2017-02-22 |
| GN228    | 无线充电           | 2017-03-06 |
| GN229    | 雄安新区           | 2017-04-06 |
| GN230    | 粤港澳大湾区       | 2017-04-12 |
| GN231    | 可燃冰             | 2017-05-22 |
| GN232    | 白马股             | 2017-05-24 |
| GN233    | 增持               | 2017-05-25 |
| GN238    | 煤电重组           | 2017-06-07 |
| GN237    | 保底增持           | 2017-06-07 |
| GN236    | 特色小镇           | 2017-06-07 |
| GN235    | 最小市值           | 2017-06-07 |
| GN234    | 装配式建筑         | 2017-06-07 |
| GN239    | 小金属             | 2017-06-19 |
| GN240    | MSCI概念           | 2017-06-20 |
| GN241    | 上海国资改革       | 2017-06-22 |
| GN244    | 小盘成长           | 2017-06-29 |
| GN242    | 一线龙头           | 2017-06-29 |
| GN243    | 二线龙头           | 2017-06-29 |
| GN245    | 融资融券           | 2017-07-14 |
| GN246    | 打板               | 2017-07-27 |
| GN247    | 房屋租赁           | 2017-07-27 |
| GN248    | 染料               | 2017-08-03 |
| GN249    | 维生素             | 2017-08-03 |
| GN250    | 钛白粉             | 2017-08-03 |
| GN252    | 鸡                 | 2017-08-09 |
| GN251    | 猪                 | 2017-08-09 |
| GN254    | 行业龙头           | 2017-08-23 |
| GN253    | 中小创蓝筹         | 2017-08-23 |
| GN255    | 3D传感             | 2017-09-02 |
| GN804    | 电解锰             | 2017-10-30 |
| GN807    | 煤化工             | 2017-10-30 |
| GN808    | 核级锆材           | 2017-10-30 |
| GN809    | 天然气管道         | 2017-10-30 |
| GN806    | 轮胎               | 2017-10-30 |
| GN810    | 足球               | 2017-10-30 |
| GN811    | 激光               | 2017-10-30 |
| GN805    | 重卡               | 2017-10-30 |
| GN803    | 氧化锌             | 2017-10-30 |
| GN710    | 绿色建筑           | 2017-10-30 |
| GN801    | 航空发动机         | 2017-10-30 |
| GN800    | 军工改制           | 2017-10-30 |
| GN799    | 电解铝             | 2017-10-30 |
| GN812    | 激光武器           | 2017-10-30 |
| GN798    | 智能燃气表         | 2017-10-30 |
| GN797    | 太阳能             | 2017-10-30 |
| GN796    | 电改               | 2017-10-30 |
| GN795    | 大气治理           | 2017-10-30 |
| GN794    | 抗生素             | 2017-10-30 |
| GN802    | 多晶硅             | 2017-10-30 |
| GN813    | 食品追溯           | 2017-10-30 |
| GN814    | 医疗改革           | 2017-10-30 |
| GN815    | 银行               | 2017-10-30 |
| GN835    | 汽车制造           | 2017-10-30 |
| GN836    | 汽配               | 2017-10-30 |
| GN838    | 碳纳米管           | 2017-10-30 |
| GN839    | 新型建材           | 2017-10-30 |
| GN840    | 高速公路           | 2017-10-30 |
| GN841    | 铝                 | 2017-10-30 |
| GN834    | 期货               | 2017-10-30 |
| GN842    | 轧板               | 2017-10-30 |
| GN844    | 钾离子电池         | 2017-10-30 |
| GN845    | 干细胞             | 2017-10-30 |
| GN846    | 抗辐射（废弃）     | 2017-10-30 |
| GN847    | 种业               | 2017-10-30 |
| GN848    | 铝空气电池         | 2017-10-30 |
| GN849    | 通信设备           | 2017-10-30 |
| GN843    | 人民币升值         | 2017-10-30 |
| GN833    | 柔性显示屏         | 2017-10-30 |
| GN832    | 农机               | 2017-10-30 |
| GN831    | 水稻               | 2017-10-30 |
| GN816    | 磷酸铁锂电池       | 2017-10-30 |
| GN817    | 太阳能电池         | 2017-10-30 |
| GN818    | 快递               | 2017-10-30 |
| GN819    | 中国版401K         | 2017-10-30 |
| GN820    | 生活垃圾处理       | 2017-10-30 |
| GN821    | 六氟磷酸锂         | 2017-10-30 |
| GN822    | 螺纹钢             | 2017-10-30 |
| GN823    | 煤炭               | 2017-10-30 |
| GN824    | 石墨烯电池         | 2017-10-30 |
| GN825    | NFC                | 2017-10-30 |
| GN826    | 镁                 | 2017-10-30 |
| GN827    | 民营金融           | 2017-10-30 |
| GN828    | 光伏               | 2017-10-30 |
| GN829    | 南方电网           | 2017-10-30 |
| GN830    | 南海               | 2017-10-30 |
| GN793    | 固废处理           | 2017-10-30 |
| GN792    | 电网混合所有制     | 2017-10-30 |
| GN791    | 城市管网           | 2017-10-30 |
| GN790    | 储能               | 2017-10-30 |
| GN739    | 光伏玻璃           | 2017-10-30 |
| GN740    | 达菲               | 2017-10-30 |
| GN741    | 广州国企           | 2017-10-30 |
| GN742    | 贵金属             | 2017-10-30 |
| GN743    | 光电子材料         | 2017-10-30 |
| GN744    | 航运               | 2017-10-30 |
| GN738    | 自然语言处理       | 2017-10-30 |
| GN745    | 医保               | 2017-10-30 |
| GN747    | 海岛整治修复       | 2017-10-30 |
| GN748    | 海洋工程           | 2017-10-30 |
| GN749    | 天水经济区         | 2017-10-30 |
| GN750    | 化学制药           | 2017-10-30 |
| GN751    | 能源管道           | 2017-10-30 |
| GN752    | 混改               | 2017-10-30 |
| GN746    | 阶梯电价           | 2017-10-30 |
| GN737    | 宽带中国           | 2017-10-30 |
| GN736    | 工程机械           | 2017-10-30 |
| GN735    | 印染               | 2017-10-30 |
| GN720    | 物流电商平台       | 2017-10-30 |
| GN721    | 电线电缆           | 2017-10-30 |
| GN722    | 东北国企           | 2017-10-30 |
| GN723    | 供气               | 2017-10-30 |
| GN724    | 节能照明           | 2017-10-30 |
| GN725    | 非晶合金材料       | 2017-10-30 |
| GN726    | 航母钢材           | 2017-10-30 |
| GN727    | 钢铁               | 2017-10-30 |
| GN728    | 港口               | 2017-10-30 |
| GN729    | 铁路运营           | 2017-10-30 |
| GN730    | 核电通风与空气处理 | 2017-10-30 |
| GN731    | 三星               | 2017-10-30 |
| GN732    | 金融IC卡           | 2017-10-30 |
| GN733    | 房地产             | 2017-10-30 |
| GN734    | 铜冶炼             | 2017-10-30 |
| GN753    | 旅游               | 2017-10-30 |
| GN850    | 三农               | 2017-10-30 |
| GN754    | 电力环保设备       | 2017-10-30 |
| GN756    | 海洋渔业           | 2017-10-30 |
| GN776    | 海南国际旅游岛     | 2017-10-30 |
| GN777    | ETC                | 2017-10-30 |
| GN778    | 焦煤               | 2017-10-30 |
| GN779    | 二手车             | 2017-10-30 |
| GN780    | 券商               | 2017-10-30 |
| GN781    | 手机游戏           | 2017-10-30 |
| GN775    | 甲型流感           | 2017-10-30 |
| GN782    | 影视传媒           | 2017-10-30 |
| GN784    | 航空               | 2017-10-30 |
| GN785    | 锂矿               | 2017-10-30 |
| GN786    | 铜                 | 2017-10-30 |
| GN787    | TIMU芯片           | 2017-10-30 |
| GN788    | 二维码             | 2017-10-30 |
| GN789    | 覆铜板             | 2017-10-30 |
| GN783    | 土地储备           | 2017-10-30 |
| GN774    | 甲醇               | 2017-10-30 |
| GN773    | 家庭互联网         | 2017-10-30 |
| GN772    | 家禽养殖           | 2017-10-30 |
| GN757    | 铁路设备           | 2017-10-30 |
| GN758    | 油品升级           | 2017-10-30 |
| GN759    | 航空航天           | 2017-10-30 |
| GN760    | 航母动力装置       | 2017-10-30 |
| GN761    | 地铁               | 2017-10-30 |
| GN762    | 无人零售           | 2017-10-30 |
| GN763    | 除尘               | 2017-10-30 |
| GN764    | 危险废物治理       | 2017-10-30 |
| GN765    | 环境监测           | 2017-10-30 |
| GN766    | 机床               | 2017-10-30 |
| GN767    | 振兴东北           | 2017-10-30 |
| GN768    | 集装箱             | 2017-10-30 |
| GN769    | 牛羊肉             | 2017-10-30 |
| GN770    | 智能驾驶（废弃）   | 2017-10-30 |
| GN771    | 语音识别           | 2017-10-30 |
| GN755    | 肉制品             | 2017-10-30 |
| GN851    | ATL电池            | 2017-10-30 |
| GN852    | 三元锂电池         | 2017-10-30 |
| GN853    | 集体建设用地       | 2017-10-30 |
| GN947    | 家居建材           | 2017-10-30 |
| GN948    | 网络视频           | 2017-10-30 |
| GN949    | 仪电仪表           | 2017-10-30 |
| GN950    | 安邦举牌           | 2017-10-30 |
| GN951    | 生物燃料           | 2017-10-30 |
| GN952    | 射频识别           | 2017-10-30 |
| GN946    | 移动健康           | 2017-10-30 |
| GN953    | 集成电路           | 2017-10-30 |
| GN955    | 油页岩             | 2017-10-30 |
| GN956    | 有色金属           | 2017-10-30 |
| GN957    | 余热利用           | 2017-10-30 |
| GN958    | 云南国资           | 2017-10-30 |
| GN959    | 海洋经济           | 2017-10-30 |
| GN960    | 舰载机             | 2017-10-30 |
| GN954    | 焦炭               | 2017-10-30 |
| GN945    | 珠宝               | 2017-10-30 |
| GN944    | 光纤               | 2017-10-30 |
| GN943    | 广东基建           | 2017-10-30 |
| GN928    | 东航集团           | 2017-10-30 |
| GN929    | 国电集团           | 2017-10-30 |
| GN930    | 中国海运           | 2017-10-30 |
| GN931    | 中航工业           | 2017-10-30 |
| GN932    | 中核集团           | 2017-10-30 |
| GN933    | 中国黄金           | 2017-10-30 |
| GN934    | 中国建材集团       | 2017-10-30 |
| GN935    | 中国联通           | 2017-10-30 |
| GN936    | 中国铝业           | 2017-10-30 |
| GN937    | 南航集团           | 2017-10-30 |
| GN938    | 西电集团           | 2017-10-30 |
| GN939    | 中国盐业总公司     | 2017-10-30 |
| GN940    | 中煤集团           | 2017-10-30 |
| GN941    | 电商物流           | 2017-10-30 |
| GN942    | 金改               | 2017-10-30 |
| GN961    | 炭素               | 2017-10-30 |
| GN927    | 中国一汽           | 2017-10-30 |
| GN962    | 金融云             | 2017-10-30 |
| GN964    | 陕甘宁             | 2017-10-30 |
| GN984    | 手势控制（废弃）   | 2017-10-30 |
| GN985    | 酒店餐饮           | 2017-10-30 |
| GN986    | 镁电池             | 2017-10-30 |
| GN987    | 操作系统国产化     | 2017-10-30 |
| GN988    | 凭祥自贸区         | 2017-10-30 |
| GN989    | 水利设备           | 2017-10-30 |
| GN983    | 客车               | 2017-10-30 |
| GN990    | 中药               | 2017-10-30 |
| GN992    | 中字头             | 2017-10-30 |
| GN993    | 重金属治理         | 2017-10-30 |
| GN994    | AMC                | 2017-10-30 |
| GN995    | 轨道交通           | 2017-10-30 |
| GN996    | 钒钛               | 2017-10-30 |
| GN997    | 液晶面板           | 2017-10-30 |
| GN991    | 中原经济区         | 2017-10-30 |
| GN982    | 瓦楞纸箱           | 2017-10-30 |
| GN981    | 中国铁总           | 2017-10-30 |
| GN980    | 上海金改           | 2017-10-30 |
| GN965    | 世界杯             | 2017-10-30 |
| GN966    | 珠三角             | 2017-10-30 |
| GN967    | 造纸               | 2017-10-30 |
| GN968    | 核电级蒸汽发生器   | 2017-10-30 |
| GN969    | 8K面板             | 2017-10-30 |
| GN970    | 河南自贸区         | 2017-10-30 |
| GN971    | 新能源客车         | 2017-10-30 |
| GN972    | 移动医疗           | 2017-10-30 |
| GN973    | 智能手机           | 2017-10-30 |
| GN974    | 互联网医疗         | 2017-10-30 |
| GN975    | 海工装备           | 2017-10-30 |
| GN976    | 钛合金（废弃）     | 2017-10-30 |
| GN977    | 太阳能逆变器       | 2017-10-30 |
| GN978    | 杭州亚运会         | 2017-10-30 |
| GN979    | 煤矿安全           | 2017-10-30 |
| GN963    | 动力电池           | 2017-10-30 |
| GN719    | 大连自贸区         | 2017-10-30 |
| GN926    | 中船重工           | 2017-10-30 |
| GN924    | 兵装集团           | 2017-10-30 |
| GN873    | 移动POS机          | 2017-10-30 |
| GN874    | 火电               | 2017-10-30 |
| GN875    | 锌二氧化锰         | 2017-10-30 |
| GN876    | 摄像头             | 2017-10-30 |
| GN877    | 煤清洁技术         | 2017-10-30 |
| GN878    | 半导体             | 2017-10-30 |
| GN872    | 锌电池             | 2017-10-30 |
| GN879    | 地沟油检测         | 2017-10-30 |
| GN881    | 铁路信息化         | 2017-10-30 |
| GN882    | 铁矿石             | 2017-10-30 |
| GN883    | 360私有化（废弃）  | 2017-10-30 |
| GN884    | 海参               | 2017-10-30 |
| GN885    | 微电子             | 2017-10-30 |
| GN886    | 混合现实           | 2017-10-30 |
| GN880    | 体外诊断           | 2017-10-30 |
| GN871    | 数字货币           | 2017-10-30 |
| GN870    | 水产               | 2017-10-30 |
| GN869    | 锂电池正负极材料   | 2017-10-30 |
| GN854    | 陕西国资           | 2017-10-30 |
| GN855    | 海上风电           | 2017-10-30 |
| GN856    | 上海国企           | 2017-10-30 |
| GN857    | 钨                 | 2017-10-30 |
| GN858    | 参股恒大           | 2017-10-30 |
| GN859    | LED液晶电视        | 2017-10-30 |
| GN860    | 生态环境保护       | 2017-10-30 |
| GN861    | 太阳能硅晶电池     | 2017-10-30 |
| GN862    | 石墨               | 2017-10-30 |
| GN863    | 煤制烯烃           | 2017-10-30 |
| GN864    | 视频监控           | 2017-10-30 |
| GN865    | 钻井平台           | 2017-10-30 |
| GN866    | 机场               | 2017-10-30 |
| GN867    | 环首都经济圈       | 2017-10-30 |
| GN868    | 远东自由经济区     | 2017-10-30 |
| GN887    | 小程序             | 2017-10-30 |
| GN925    | 中船集团           | 2017-10-30 |
| GN888    | 天然气             | 2017-10-30 |
| GN890    | 农资流通           | 2017-10-30 |
| GN910    | 国产芯片           | 2017-10-30 |
| GN911    | 融资租赁           | 2017-10-30 |
| GN912    | 海岛监视监测       | 2017-10-30 |
| GN913    | 沙尘暴             | 2017-10-30 |
| GN914    | 新疆兵团国资       | 2017-10-30 |
| GN915    | 信托               | 2017-10-30 |
| GN909    | 参股消费金融       | 2017-10-30 |
| GN916    | 无效4              | 2017-10-30 |
| GN918    | 畜牧业             | 2017-10-30 |
| GN919    | LNG汽车            | 2017-10-30 |
| GN920    | 宝武钢铁集团       | 2017-10-30 |
| GN921    | 神华集团           | 2017-10-30 |
| GN922    | 国家电网           | 2017-10-30 |
| GN923    | 中国兵工集团       | 2017-10-30 |
| GN917    | 参股河北企业       | 2017-10-30 |
| GN908    | 海上丝绸之路       | 2017-10-30 |
| GN907    | 险资举牌           | 2017-10-30 |
| GN906    | 支付宝             | 2017-10-30 |
| GN891    | 蓄电池             | 2017-10-30 |
| GN892    | 特钢               | 2017-10-30 |
| GN893    | 储气设施           | 2017-10-30 |
| GN894    | 锂电池电解液       | 2017-10-30 |
| GN895    | 国产手机           | 2017-10-30 |
| GN896    | 三维地图           | 2017-10-30 |
| GN897    | 西气东输           | 2017-10-30 |
| GN898    | 铅锌金属           | 2017-10-30 |
| GN899    | 无人驾驶（废弃）   | 2017-10-30 |
| GN900    | 西安国资           | 2017-10-30 |
| GN901    | 手机芯片           | 2017-10-30 |
| GN902    | 西藏国资           | 2017-10-30 |
| GN903    | 西电东送           | 2017-10-30 |
| GN904    | 西沙群岛           | 2017-10-30 |
| GN905    | 稀有金属           | 2017-10-30 |
| GN889    | 船舶               | 2017-10-30 |
| GN718    | 土壤环保           | 2017-10-30 |
| GN999    | 染料涂料           | 2017-10-30 |
| GN716    | 磁悬浮列车         | 2017-10-30 |
| GN659    | 虚拟运营商         | 2017-10-30 |
| GN660    | 芯片替代           | 2017-10-30 |
| GN661    | 脱硫脱硝           | 2017-10-30 |
| GN662    | 乳业               | 2017-10-30 |
| GN663    | 东北亚经贸中心     | 2017-10-30 |
| GN664    | 铝材加工           | 2017-10-30 |
| GN665    | 小米产业链         | 2017-10-30 |
| GN658    | 烟气脱硫           | 2017-10-30 |
| GN666    | 杀虫剂             | 2017-10-30 |
| GN668    | 配电网             | 2017-10-30 |
| GN669    | 智能建筑           | 2017-10-30 |
| GN670    | 园林               | 2017-10-30 |
| GN671    | 外贸               | 2017-10-30 |
| GN672    | 水处理膜           | 2017-10-30 |
| GN673    | 国企改革           | 2017-10-30 |
| GN674    | 陆基反导           | 2017-10-30 |
| GN667    | 铁路运价改革       | 2017-10-30 |
| GN657    | 甲流H3N2           | 2017-10-30 |
| GN656    | 再生金属           | 2017-10-30 |
| GN655    | 饲料               | 2017-10-30 |
| GN717    | 成品油             | 2017-10-30 |
| GN998    | 水泥               | 2017-10-30 |
| GN639    | 参股券商           | 2017-10-30 |
| GN640    | 人脸识别           | 2017-10-30 |
| GN641    | 出租车             | 2017-10-30 |
| GN643    | 传媒               | 2017-10-30 |
| GN644    | 核电主设备         | 2017-10-30 |
| GN645    | 民营宽带           | 2017-10-30 |
| GN646    | 保险               | 2017-10-30 |
| GN647    | 粮食安全           | 2017-10-30 |
| GN648    | 黄金               | 2017-10-30 |
| GN649    | 精准医疗           | 2017-10-30 |
| GN650    | 自动售货机         | 2017-10-30 |
| GN651    | 铁路配件           | 2017-10-30 |
| GN652    | 循环经济           | 2017-10-30 |
| GN653    | 鸡肉               | 2017-10-30 |
| GN654    | 百草枯             | 2017-10-30 |
| GN675    | 互联网汽车         | 2017-10-30 |
| GN676    | 东亚自贸           | 2017-10-30 |
| GN642    | 白银               | 2017-10-30 |
| GN678    | 水环境修复         | 2017-10-30 |
| GN699    | 金属新材料         | 2017-10-30 |
| GN700    | P2P                | 2017-10-30 |
| GN701    | 阿里巴巴概念       | 2017-10-30 |
| GN702    | 物流骨干网         | 2017-10-30 |
| GN703    | 物流               | 2017-10-30 |
| GN704    | 卷烟纸             | 2017-10-30 |
| GN705    | 白酒               | 2017-10-30 |
| GN698    | 人民币贬值         | 2017-10-30 |
| GN706    | 家电               | 2017-10-30 |
| GN709    | 打车软件           | 2017-10-30 |
| GN711    | 中央政务区         | 2017-10-30 |
| GN712    | 地热               | 2017-10-30 |
| GN713    | 油气采掘           | 2017-10-30 |
| GN714    | 草甘膦             | 2017-10-30 |
| GN715    | 车联网             | 2017-10-30 |
| GN677    | 医疗器械           | 2017-10-30 |
| GN707    | 超市               | 2017-10-30 |
| GN697    | 铁路机车           | 2017-10-30 |
| GN708    | 北斗导航           | 2017-10-30 |
| GN695    | 游戏直播           | 2017-10-30 |
| GN679    | 涉农改革           | 2017-10-30 |
| GN680    | 绿色经济           | 2017-10-30 |
| GN696    | 地下管廊           | 2017-10-30 |
| GN682    | 猪肉               | 2017-10-30 |
| GN683    | 核电阀门           | 2017-10-30 |
| GN684    | 核电级压力容器     | 2017-10-30 |
| GN685    | 大飞机             | 2017-10-30 |
| GN686    | 防汛               | 2017-10-30 |
| GN681    | 智慧停车           | 2017-10-30 |
| GN692    | LED                | 2017-10-30 |
| GN688    | 农村金融           | 2017-10-30 |
| GN693    | 碳纤维             | 2017-10-30 |
| GN689    | 汽车销售           | 2017-10-30 |
| GN690    | 农田水利建设       | 2017-10-30 |
| GN691    | 纤维新材料         | 2017-10-30 |
| GN694    | 电子政务           | 2017-10-30 |
| GN687    | 动力煤             | 2017-10-30 |
| GN256    | 360概念            | 2017-11-08 |
| GN257    | 可转债             | 2017-11-14 |
| GN258    | 新型城镇化         | 2017-12-29 |
| GN1000   | 金融科技           | 2018-02-05 |
| GN1001   | ERP软件            | 2018-02-06 |
| GN1050   | 智能音箱           | 2018-02-07 |
| GN1038   | 电动物流车         | 2018-02-07 |
| GN1047   | 音乐产业           | 2018-02-07 |
| GN1051   | 液态金属           | 2018-02-07 |
| GN1046   | 聚碳酸酯           | 2018-02-07 |
| GN1039   | 氟化工             | 2018-02-07 |
| GN1048   | K-12教育           | 2018-02-07 |
| GN1055   | 独家药             | 2018-02-07 |
| GN1091   | 海南省国资         | 2018-02-07 |
| GN1163   | 大消费             | 2018-02-07 |
| GN1176   | 调味品             | 2018-02-07 |
| GN1201   | 糖产业             | 2018-02-07 |
| GN1086   | 中概股回归         | 2018-02-07 |
| GN1066   | 天津市国资         | 2018-02-07 |
| GN1067   | 河北省国资         | 2018-02-07 |
| GN1068   | 南京市国资         | 2018-02-07 |
| GN1056   | 血液制品           | 2018-02-07 |
| GN1028   | 长三角             | 2018-02-07 |
| GN1013   | 国产软硬件         | 2018-02-07 |
| GN1083   | 钴                 | 2018-02-07 |
| GN1011   | 深圳市国资         | 2018-02-07 |
| GN1002   | 乡村振兴           | 2018-02-07 |
| GN1017   | 广东省国资         | 2018-02-07 |
| GN1156   | 台资概念           | 2018-02-07 |
| GN1069   | 江苏省国资         | 2018-02-07 |
| GN1003   | 万达私有化         | 2018-02-08 |
| GN1004   | 腾讯概念           | 2018-02-08 |
| GN1009   | 相变存储           | 2018-02-09 |
| GN1005   | 吉利概念           | 2018-02-26 |
| GN1006   | eID                | 2018-03-01 |
| GN1007   | 华为               | 2018-03-01 |
| GN1010   | 物产系             | 2018-03-01 |
| GN1012   | 雄安地热           | 2018-03-01 |
| GN1008   | 王亚伟             | 2018-03-01 |
| GN1014   | 工业互联网         | 2018-03-02 |
| GN1015   | 狂犬病             | 2018-03-02 |
| GN1016   | 北京市国资         | 2018-03-02 |
| GN1020   | 富士康概念         | 2018-03-05 |
| GN1021   | 独角兽概念         | 2018-03-05 |
| GN1019   | 参股宁德时代       | 2018-03-05 |
| GN1018   | 蚂蚁金服           | 2018-03-05 |
| GN1022   | 全面屏             | 2018-03-06 |
| GN1023   | eSIM卡             | 2018-03-06 |
| GN1024   | 国产化创新         | 2018-03-06 |
| GN1026   | 3D玻璃             | 2018-03-07 |
| GN1025   | 量子计算机         | 2018-03-07 |
| GN1027   | 炭黑               | 2018-03-07 |
| GN1029   | 抗艾滋病           | 2018-03-08 |
| GN1030   | 烧碱               | 2018-03-12 |
| GN1031   | 无屏电视           | 2018-03-12 |
| GN1032   | 有机硅             | 2018-03-13 |
| GN1033   | 钾肥               | 2018-03-13 |
| GN1035   | 西部水泥           | 2018-03-13 |
| GN1036   | 保理业务           | 2018-03-13 |
| GN1034   | 宁德时代产业链     | 2018-03-13 |
| GN1040   | CAR-T疗法          | 2018-03-14 |
| GN1037   | 萤石               | 2018-03-14 |
| GN1041   | 废品回收利用       | 2018-03-14 |
| GN1043   | 玻璃纤维           | 2018-03-15 |
| GN1042   | 玻璃               | 2018-03-15 |
| GN1044   | 啤酒               | 2018-03-16 |
| GN1045   | 黄酒               | 2018-03-16 |
| GN1049   | 创新药             | 2018-03-19 |
| GN1052   | Micro LED          | 2018-03-20 |
| GN1053   | QLED               | 2018-03-20 |
| GN1054   | 供应链金融         | 2018-03-20 |
| GN1058   | 服务器             | 2018-03-21 |
| GN1062   | 工业气体           | 2018-03-21 |
| GN1057   | 血液透析           | 2018-03-21 |
| GN1063   | 氢能               | 2018-03-22 |
| GN1064   | 数字中国           | 2018-03-22 |
| GN1072   | 反关税             | 2018-03-23 |
| GN1065   | 制冷剂             | 2018-03-23 |
| GN1070   | 出口型企业         | 2018-03-23 |
| GN1071   | 地方国企           | 2018-03-23 |
| GN1073   | 机电产品           | 2018-03-23 |
| GN1074   | 无感支付           | 2018-03-26 |
| GN1078   | 苯酚               | 2018-03-26 |
| GN1075   | 医药连锁           | 2018-03-26 |
| GN1076   | 参股药明康德       | 2018-03-26 |
| GN1077   | 无缝钢管           | 2018-03-26 |
| GN1080   | 南北船合并         | 2018-03-27 |
| GN1079   | 挖掘机             | 2018-03-27 |
| GN1081   | 机器视觉           | 2018-03-29 |
| GN1082   | 盐湖提锂           | 2018-03-29 |
| GN1085   | CDR概念            | 2018-04-02 |
| GN1084   | 室内导航           | 2018-04-02 |
| GN1088   | 崇明生态岛         | 2018-04-04 |
| GN1087   | 仿制药             | 2018-04-04 |
| GN1089   | 稀土               | 2018-04-09 |
| GN1092   | 合资企业           | 2018-04-11 |
| GN1093   | 知识产权           | 2018-04-11 |
| GN1094   | 金融开放           | 2018-04-13 |
| GN1095   | 纯碱               | 2018-04-13 |
| GN1096   | 硅晶圆             | 2018-04-17 |
| GN1097   | 云通信             | 2018-04-17 |
| GN1098   | 高送转预期         | 2018-04-17 |
| GN1099   | 领涨龙头           | 2018-04-19 |
| GN1100   | 蔚来汽车           | 2018-04-24 |
| GN1101   | 沪深港通重仓       | 2018-04-25 |
| GN1102   | 波音公司           | 2018-04-26 |
| GN1103   | 扭亏               | 2018-05-02 |
| GN1104   | 超导               | 2018-05-03 |
| GN1105   | 大豆               | 2018-05-04 |
| GN1106   | 摘帽               | 2018-05-08 |
| GN1107   | 新股               | 2018-05-16 |
| GN1108   | 棉花               | 2018-05-17 |
| GN1121   | 广东自贸区         | 2018-05-23 |
| GN1120   | 陕西自贸区         | 2018-05-23 |
| GN1119   | 四川自贸区         | 2018-05-23 |
| GN1118   | 养老金概念         | 2018-05-23 |
| GN1109   | 化妆品             | 2018-05-23 |
| GN1116   | 重庆自贸区         | 2018-05-23 |
| GN1115   | 北京冬奥会         | 2018-05-23 |
| GN1114   | 超级细菌           | 2018-05-23 |
| GN1113   | 电子车牌           | 2018-05-23 |
| GN1112   | 沪伦通             | 2018-05-23 |
| GN1110   | 油价上调           | 2018-05-23 |
| GN1117   | 智慧林业           | 2018-05-23 |
| GN1111   | 钢铁电商           | 2018-05-23 |
| GN1122   | 电阻               | 2018-05-24 |
| GN1123   | 纳米               | 2018-05-24 |
| GN1124   | 葡萄酒             | 2018-05-30 |
| GN1125   | 家纺               | 2018-05-30 |
| GN1126   | 破净               | 2018-05-31 |
| GN1127   | 云存储             | 2018-06-05 |
| GN1128   | 电子纸             | 2018-06-06 |
| GN1130   | 触板               | 2018-06-13 |
| GN1129   | 抖音概念           | 2018-06-13 |
| GN1131   | 二极管             | 2018-06-15 |
| GN1132   | 百度概念           | 2018-06-15 |
| GN1133   | 成交主力           | 2018-06-21 |
| GN1135   | 低价股             | 2018-06-25 |
| GN1134   | 高价股             | 2018-06-25 |
| GN1136   | 品牌龙头           | 2018-07-04 |
| GN1137   | 股票回购           | 2018-07-09 |
| GN1138   | 汇金概念           | 2018-07-10 |
| GN1139   | 聚酰亚胺           | 2018-07-18 |
| GN1140   | 富勒烯             | 2018-07-18 |
| GN1159   | 社保重仓           | 2018-07-19 |
| GN1141   | 基金重仓           | 2018-07-19 |
| GN1142   | 超级高铁           | 2018-07-20 |
| GN1143   | Facebook概念       | 2018-07-25 |
| GN1145   | 西部大基建         | 2018-08-01 |
| GN1144   | 中国国际进口博览会 | 2018-08-01 |
| GN1147   | 宠物食品           | 2018-08-03 |
| GN1146   | 谷歌概念           | 2018-08-03 |
| GN1148   | 无效82             | 2018-08-06 |
| GN1149   | 大基建央企         | 2018-08-10 |
| GN1150   | 网络可视化         | 2018-08-13 |
| GN1151   | 多肽药             | 2018-08-15 |
| GN1152   | 双百企业           | 2018-08-15 |
| GN1153   | PTA                | 2018-08-21 |
| GN1154   | 锦纶               | 2018-08-22 |
| GN1155   | 涤纶               | 2018-08-22 |
| GN1157   | 子弹短信           | 2018-08-30 |
| GN1158   | QFII重仓           | 2018-08-30 |
| GN1160   | 月饼               | 2018-09-10 |
| GN1161   | 中非合作概念       | 2018-09-11 |
| GN1162   | 尿素               | 2018-09-14 |
| GN1164   | 电路板             | 2018-09-26 |
| GN1165   | 富时罗素概念       | 2018-09-27 |
| GN1166   | 中远海运集团       | 2018-10-08 |
| GN1167   | 制造业单项冠军企业 | 2018-10-09 |
| GN1168   | 宝马概念           | 2018-10-16 |
| GN1169   | 系统重要性金融机构 | 2018-10-16 |
| GN1170   | 国资入股           | 2018-10-18 |
| GN1171   | 低价小市值         | 2018-11-13 |
| GN1172   | 民营企业           | 2018-11-13 |
| GN1173   | 广电系             | 2018-11-28 |
| GN1174   | 标普道琼斯概念     | 2018-12-07 |
| GN1175   | 券商重仓           | 2018-12-24 |
| GN1177   | 信托重仓           | 2019-01-14 |
| GN1178   | 私募重仓           | 2019-01-16 |
| GN1179   | 磷化工             | 2019-01-18 |
| GN1180   | 一号文件           | 2019-01-22 |
| GN1181   | 生态农业           | 2019-01-22 |
| GN1182   | 供销社             | 2019-01-28 |
| GN1183   | 业绩爆雷           | 2019-01-30 |
| GN1184   | 纳米银             | 2019-02-21 |
| GN1185   | 工业大麻           | 2019-02-22 |
| GN1186   | 生物质能           | 2019-03-01 |
| GN1187   | 边缘计算           | 2019-03-04 |
| GN1188   | 参股科创板         | 2019-03-05 |
| GN1189   | MSCI中盘           | 2019-03-06 |
| GN1190   | 纸电池             | 2019-03-08 |
| GN1191   | 泛在电力物联网     | 2019-03-11 |
| GN1192   | 数字孪生           | 2019-03-13 |
| GN1193   | 网络切片           | 2019-03-14 |
| GN1194   | 电子烟             | 2019-03-18 |
| GN1195   | 西安自贸区         | 2019-03-21 |
| GN1196   | 安全生产           | 2019-03-25 |
| GN1197   | 流媒体             | 2019-03-26 |
| GN1199   | 参股外资券商       | 2019-04-01 |
| GN1198   | 国美系             | 2019-04-01 |
| GN1200   | 横琴旅游岛         | 2019-04-02 |
| GN1202   | 长三角自贸区       | 2019-04-11 |
| GN1203   | 醋酸               | 2019/4/15  |
| GN1204   | 宁夏基建           | 2019/4/15  |
| GN1205   | 陆股通日买入前二十 | 2019/4/16  |
| GN1206   | 陆股通日卖出前二十 | 2019/4/18  |
| GN1207   | 陆股通日增仓前二十 | 2019/4/18  |
| GN1208   | 陆股通日减仓前二十 | 2019/4/18  |
| GN1209   | 陆股通周卖出前二十 | 2019/4/19  |
| GN1217   | 陆股通周买入前二十 | 2019/4/19  |
| GN1222   | 陆股通周增仓前二十 | 2019/4/22  |
| GN1213   | 陆股通周减仓前二十 | 2019/4/22  |
| GN1215   | 陆股通月买入前二十 | 2019/4/22  |
| GN1220   | 陆股通月卖出前二十 | 2019/4/22  |
| GN1212   | 陆股通月减仓前二十 | 2019/4/23  |
| GN1210   | 陆股通月增仓前二十 | 2019/4/23  |
| GN1214   | 陆股通持续净买入   | 2019/4/24  |
| GN1219   | 陆股通持续净卖出   | 2019/4/24  |
| GN1211   | 陆股通增仓         | 2019/4/25  |
| GN1221   | 陆股通减仓         | 2019/4/25  |
| GN1216   | 陆股通建仓         | 2019/4/26  |
| GN1218   | 陆股通清仓         | 2019/4/26  |
| GN1223   | 机构大额买入       | 2019/4/28  |
| GN1224   | 机构大额卖出       | 2019/4/28  |

# 获取行业指数数据

## 获取申万一级行业日行情数据

from jqdatasdk import finance  
df=finance.run_query(query(finance.SW1_DAILY_PRICE).filter(finance.SW1_DAILY_PRICE.code=='801010').limit(n))  
print(df)  
记录了申万一级行业指数的历史日行情数据，每日18:00更新。  

**参数：**

query(finance.SW1_DAILY_PRICE)：表示从finance.SW1_DAILY_PRICE这张表中查询申万一级行业指数的历史日行情数据，还可以指定所要查询的字段名，格式如下：query(库名.表名.字段名1，库名.表名.字段名2），多个字段用英文逗号分隔进行提取；如query(finance.SW1_DAILY_PRICE.code)。query函数的更多用法详见：query简易教程。  
finance.SW1_DAILY_PRICE：代表申万一级行业日行情数据表，收录了申万一级行业指数的历史日行情数据，表结构和字段信息如下：

| 字段名称   | 中文名称   | 字段类型      | 能否为空 | 注释                           |
| ---------- | ---------- | ------------- | -------- | ------------------------------ |
| date       | 交易日     | date          | N        |                                |
| code       | 指数编码   | varchar(12)   | N        | 对应申万一级行业指数编码       |
| name       | 指数名称   | varchar(20)   | N        |                                |
| open       | 开盘指数   | decimal(20,4) |          |                                |
| high       | 最高指数   | decimal(20,4) |          |                                |
| low        | 最低指数   | decimal(20,4) |          |                                |
| close      | 收盘指数   | decimal(20,4) |          |                                |
| pre_close  | 昨收盘指数 | decimal(20,4) |          | 昨收盘指数用昨日的收盘指数填充 |
| volume     | 成交量     | decimal(20,4) |          | 单位：股                       |
| money      | 成交额     | decimal(20,4) |          | 单位：元                       |
| change_pct | 涨跌幅     | decimal(10,4) |          | 单位：％                       |

 

filter(finance.SW1_DAILY_PRICE.code==801010)：指定筛选条件，通过finance.SW1_DAILY_PRICE.code==801010可以查询指定行业（如农林牧渔801010）的日行情数据；除此之外，还可以对表中其他字段指定筛选条件，如finance.SW1_DAILY_PRICE.date=='2019-03-01'，指定获取2019年3月1日的申万一级行业日行情数据。  
limit(n)：限制返回的数据条数，n指定返回条数。  

**返回结果：**

返回一个 dataframe，每一行对应数据表中的一条数据， 列索引是您所查询的字段名称

**注意：**

1. 为了防止返回数据量过大, 我们每次最多返回3000行
2. 不能进行连表查询，即同时查询多张表的数据  

**示例：**

In [23]:
# 查询最近10个交易日申万一级行业指数-农林牧渔行业（801010）的日行情数据。
# from jqdatasdk import finance
df=finance.run_query(query(finance.SW1_DAILY_PRICE).filter(finance.SW1_DAILY_PRICE.code=='801010').order_by(finance.SW1_DAILY_PRICE.date.desc()).limit(10))
print(df.head())

        id        date    code  name     open     high      low    close  \
0  1782018  2019-07-24  801010  农林牧渔  3390.01  3420.87  3375.22  3403.73   
1  1781486  2019-07-23  801010  农林牧渔  3357.65  3386.37  3344.67  3378.28   
2  1780954  2019-07-22  801010  农林牧渔  3448.30  3448.30  3351.57  3354.49   
3  1780422  2019-07-19  801010  农林牧渔  3435.11  3486.11  3433.94  3463.47   
4  1779890  2019-07-18  801010  农林牧渔  3441.27  3476.06  3429.01  3429.01   

         volume         money  change_pct  
0  1.230905e+09  1.294780e+10        0.75  
1  8.635670e+08  1.077070e+10        0.71  
2  1.362378e+09  1.609689e+10       -3.15  
3  1.324605e+09  1.517428e+10        1.00  
4  1.506355e+09  1.817348e+10       -1.00  


## 获取申万一级行业估值数据
from jqdatasdk import finance  
df=finance.run_query(query(finance.SW1_DAILY_VALUATION).filter(finance.SW1_DAILY_VALUATION.code=='801010').limit(n))  
print(df)  
记录了申万一级行业指数的每日估值数据，每天18:00更新。  

**参数：**

query(finance.SW1_DAILY_VALUATION)：表示从finance.SW1_DAILY_VALUATION这张表中查询申万一级行业指数的估值数据，还可以指定所要查询的字段名，格式如下：query(库名.表名.字段名1，库名.表名.字段名2），多个字段用英文逗号分隔进行提取；如query(finance.SW1_DAILY_VALUATION.code)。query函数的更多用法详见：query简易教程。  
finance.SW1_DAILY_VALUATION：代表申万一级行业指数估值表，收录了申万一级行业指数的历史估值数据，表结构和字段信息如下：

| 字段名称                       | 中文名称     | 字段类型      | 能否为空 | 注释                                                         |
| ------------------------------ | ------------ | ------------- | -------- | ------------------------------------------------------------ |
| date                           | 交易日       | date          | N        |                                                              |
| code                           | 指数编码     | varchar(12)   | N        | 对应申万一级行业指数编码                                     |
| name                           | 指数名称     | varchar(20)   | N        |                                                              |
| turnover_ratio                 | 换手率       | decimal(10,4) |          | 单位：％                                                     |
| pe                             | 市盈率       | decimal(20,4) |          | 单位：倍；PE = 流通市值／最近4个季度的净利润；最近 4 个季度的净利润按如下方法计算: 1-4 月,最近 4 个季度的净利润=上一年度前 3 季度累计净利润+上上一年度的四季度净利润；5-8 月,最近 4 个季度的净利润=当年 1 季度净利润+前 1 年净利润-前 1 年 1 季度净利润；9-10 月,最近 4 个季度的净利润=当年中期净利润+前 1 年净利润-前 1 年中期净利润；11-12 月,最近 4 个季度的净利润=当年前 3 季度累计净利润+上1年年度净利润-上 1 年前 3 季度累计净利润 |
| pb                             | 市净率       | decimal(20,4) |          | 单位：倍；按照自由流通量加权的净资产倍率。 PB = 流通市值／按照流通市值计算的净资产；按照流通市值计算的净资产 ＝ 最新净资产*流通股本／总股本 |
| average_price                  | 均价         | decimal(20,4) |          | 单位：元。指数成份股在统计期最后交易日收盘的简单算术平均价   |
| money_ratio                    | 成交额占比   | decimal(10,4) |          | 单位：％；成交额占比＝某个行业成交额／所有行业成交额之和     |
| circulating_market_cap         | 流通市值     | decimal(20,4) |          | 单位：元                                                     |
| average_circulating_market_cap | 平均流通市值 | decimal(20,4) |          | 单位：元；平均流通市值＝流通市值／所在行业股票数             |
| dividend_ratio                 | 股息率       | decimal(10,4) |          | 单位：％；按照自由流通量加权的现金股息率；dividend_ratio=Df/Vf；Df: 所有股票在截止日的一 个自然年(365日)中所累积派发的税前现金红利之和按照流通股本对应的分红量；Vf: 该行业成分股股的流通市值之和 |

filter(finance.SW1_DAILY_VALUATION.code==801010)：指定筛选条件，通过finance.SW1_DAILY_VALUATION.code=='801010'可以查询指定行业（如农林牧渔801010）的估值数据；除此之外，还可以对表中其他字段指定筛选条件，如finance.SW1_DAILY_VALUATION.date=='2019-03-01'，指定获取2019年3月1日的申万一级行业的估值数据。  
limit(n)：限制返回的数据条数，n指定返回条数。  

**返回结果：**

返回一个 dataframe，每一行对应数据表中的一条数据， 列索引是您所查询的字段名称

**注意：**

1. 为了防止返回数据量过大, 我们每次最多返回3000行
2. 不能进行连表查询，即同时查询多张表的数据

**示例：**

In [24]:
# 查询最近10个交易日申万一级行业指数-农林牧渔行业（801010）的估值数据
# from jqdatasdk import finance
df=finance.run_query(query(finance.SW1_DAILY_VALUATION).filter(finance.SW1_DAILY_VALUATION.code=='801010').order_by(finance.SW1_DAILY_VALUATION.date.desc()).limit(10))
print(df.head())

        id        date    code  name  turnover_ratio     pe    pb  \
0  1517210  2019-07-24  801010  农林牧渔          1.8149  52.21  3.95   
1  1516494  2019-07-23  801010  农林牧渔          1.2733  51.83  3.92   
2  1515778  2019-07-22  801010  农林牧渔          2.0087  51.56  3.89   
3  1515062  2019-07-19  801010  农林牧渔          1.9531  53.23  4.01   
4  1514346  2019-07-18  801010  农林牧渔          2.2213  52.71  3.97   

   average_price  money_ratio  circulating_market_cap  \
0          12.30         3.44            8.526449e+11   
1          12.18         3.59            8.462207e+11   
2          12.05         4.41            8.402183e+11   
3          12.51         4.35            8.681311e+11   
4          12.47         5.01            8.608687e+11   

   average_circulating_market_cap  dividend_ratio  
0                    1.003112e+10            0.95  
1                    9.955538e+09            0.95  
2                    9.884922e+09            0.96  
3                    1.021331e+10 